---
title: "Keynote Presentations"
---

## K1: Phenomenology for maize experts: what do you see when you contemplate the map of global maize production?

*Thursday, February 29 7:20 PM EST*

-   (submitted by Ricardo Salvador [rsalvador\@ucsusa.org](mailto:rsalvador@ucsusa.org){.email})

-   Full Author List: Salvador, Ricardo J1

-   1 Union of Concerned Scientists; Washington DC, USA 20006

::: {.callout-note collapse="true"}
## Abstract

An agricultural system is the landscape expression of a set of human values, on the view of geographer Carl Sauer (1889 - 1975.) “The material manifestations of culture are land use, settlement patterns, technology, and other artifacts.” The community of maize researchers and technologists has contributed to the creation of six global basins of agricultural hyper-productivity. The values manifested by the existence of these basins reflect prioritization of return to financial capital, industrial specialization and efficiency, at the expense of environmental degradation (global warming, soil erosion, hypoxia) and human displacement (decrease in the number, and migration, of both self-provisioning and industrial farmers.) To the extent that maize researchers intend their activities and contributions to be unqualified social goods, there is a need for this community to address the incongruity between those intentions and the actual social outcomes. The communities affected by the spread of industrial maize production, together with scientists whose expertise is understanding the grand currents and drivers of cultural development—anthropologists, historians, sociologists—can enrich the universe of considerations and evaluation of the “maize improvement” project. Funding acknowledgement: Members of the Union of Concerned Scientists
:::

## K2: Selected highlights of maize disease resistance genetics research: These are a few of my favorite things.

*Friday, March 17 11:40 AM EST*

-   (submitted by Peter Balint-Kurti [pjbalint\@ncsu.edu](mailto:pjbalint@ncsu.edu){.email})

-   Full Author List: Balint-Kurti, Peter J.1

-   1 Plant Science Research Unit, USDA-ARS, Raleigh NC, USA

::: {.callout-note collapse="true"}
## Abstract

The maize genetic system has many unique and attractive features. While, perhaps surprisingly, it has not been used as much as some other systems in molecular-genetic studies of plant disease resistance, a number of important discoveries in this area have been made using maize. The southern corn leaf blight epidemic of 1970, caused by the widespread distribution of (what turned out to be) a disease susceptibility gene, resulted in losses of \~\$8 billion (2023 dollars). Maize pathologists were instrumental in helping ensure that the epidemic lasted just a single year. This epidemic has often been cited as a cautionary tale regarding the dangers of genetic uniformity. I will propose alternative perspectives and lessons that might be drawn from this event. The first plant disease susceptibility and resistance genes were identified in maize. Studies of the maize Rp1 common rust resistance locus were important in our understanding of the complex nature of many plant resistance loci. Further work with Rp1 demonstrated the cell autonomous nature of the plant hypersensitive defense response (also known as HR). More recently, auto-active variants of Rp1 have been used to describe the molecular genetic architecture that controls HR. The excellent resources available in maize for quantitative genetic studies have enabled some of the most detailed studies describing the basis of quantitative disease resistance and multiple disease resistance. Several maize quantitative disease resistance genes have been identified over the past decade, informing our understanding of the complex basis of plant quantitative disease resistance. I will discuss these ‘selected highlights’ with some reference to our own work. I will further discuss prospects and applications of our burgeoning knowledge in this area. Funding acknowledgement: National Science Foundation (NSF), United States Department of Agriculture (USDA)
:::

## K3: From the stars to your table - plants as complex conduits for iron nutrition

*Friday, March 1 10:40 AM EST*

-   (submitted by Terri Long [terri_long\@ncsu.edu](mailto:terri_long@ncsu.edu){.email})

-   Full Author List: Long, Terri1

-   1 Dept. of Plant and Microbial Biology, NC State University, Raleigh, NC 27695

::: {.callout-note collapse="true"}
## Abstract

Iron is a ubiquitous micronutrient that plays critical roles in central metabolic processes for all living organisms. The mechanisms by which plants extract iron from soil and maintain iron homeostasis are particularly intriguing. While it is relatively abundant, in most soils iron is insoluble and therefore of limited bioavailability, however excess iron accumulation in plants can lead to cellular damage. Thus, plants must extract sufficient quantities of iron from recalcitrant soil environments, while also ensuring that iron content does not exceed a specific range. Arabidopsis and other dicots have evolved mechanisms to sense iron deficiency in the shoot, which triggers roots to solubilize, reduce and uptake iron across multiple root cell types before transport to the shoot. Using a combination of molecular and confocal microscopy analysis, cell-type specific transcriptional profiling, and mathematical modeling we have uncovered several molecular mechanisms that control how plants recognize and respond to iron deficiency stress in a root cell-specific manner. Our findings provide new evidence for how distinct alternations in the root cortex control carbon metabolism in response to iron deprivation, and how iron deficiency causes specific developmental alterations in the root vasculature and epidermis. Together, these mechanisms operate to fine-tune root growth and physiology in the face of suboptimal growth conditions, while also providing new avenues for exploring inter- and intracellular nutrient stress response in plants. Funding acknowledgement: National Science Foundation (NSF), United States Department of Agriculture (USDA)
:::

## K4: Leveraging synthetic biology to monitor and control plant hormone activity

*Saturday, March 2 10:40 AM EST*

-   (submitted by Anna Stepanova [atstepan\@ncsu.edu](mailto:atstepan@ncsu.edu){.email})

-   Full Author List: Stepanova, Anna N.1

-   1 Department of Plant and Microbial Biology, North Carolina State University, Raleigh, NC, USA

::: {.callout-note collapse="true"}
## Abstract

Phytohormones are critical regulators of plant development and environmental responses. In the past three decades, the molecular pathways that govern hormone biosynthesis, signaling, and catabolism have been largely mapped out using a combination of genetics, molecular biology, biochemistry, and cell biology approaches. Despite the major progress, our ability to monitor and precisely control hormone action remains limited. With the development of inexpensive DNA synthesis technologies and the rise of synthetic biology as a new discipline at the intersection of molecular genetics and engineering, new molecular tools can now be built to enable hormone tracking and targeted hormone manipulation. We have generated a synthetic biology toolbox that allows rapid construction of multi-hormone transcriptional reporters. In addition, we are building CRISPR-based logic gate devices to confer novel, highly restricted patterns of expression to any genes of interest using a limited set of available native and synthetic drivers. The latter technology can be employed to tune the expression levels and subtract undesired domains of expression from existing drivers to precisely control output genes of interest, such as hormone biosynthesis, signaling, or catabolism genes, to regulate plant architecture, responses to stress, and other traits of interest. By combining multi- hormone reporters and genetic logic devices, we aim to shed fresh light on the mechanistic role of hormones in orchestrating plant development and stress physiology. That knowledge can then be relied upon to develop resilient next-generation crops. Funding acknowledgement: National Science Foundation (NSF
:::

## K5: Plant breeding and the infinitesimal model: cause or consequence

*Saturday, March 2 11:40 AM EST*

-   (submitted by Natalia de Leon [ndeleongatti\@wisc.edu](mailto:ndeleongatti@wisc.edu){.email})

-   Full Author List: de Leon, Natalia1

-   1 Department of Biology, University of Wisconsin, Madison, WI, USA

::: {.callout-note collapse="true"}
## Abstract

Breeding relies on genetic variation generated through random mutagenesis and recombination and simultaneously shapes such variation by the process of selection. Starting with the domestication processes which led to the development of modern crops, to contemporary tools that refine artificial selection, large beneficial effects controlling targets of selection are expected to be fixed, limiting subsequent improvements to genes of minor effect. This limitation, however, does not result from biological constraints but rather from the allelic purge induced by selection. The practical consequence of this is that enhancing economically important traits in plants exclusively through the exploration of existing variation is unlikely to yield significant progress, particularly at the scale demanded by a growing global population. The advent of modern technologies, including genotype-agnostic transformation and editing resources, enable breeders to move away from a traditional breeding lottery approach to a situation where it is now feasible to deliberately optimize combination of alleles that can be further exploiting from a breeding context. As we consider sources of variation and methods to evaluated and harness them effectively, it is important to consider where such variation is prevalent and which techniques hold promise for generating the magnitude of changes required to meet global demand. This presentation will discuss how selection processes have likely altered variation and the ramifications for technological applications aimed at increasing the efficiency of breeding efforts. Funding acknowledgement: United States Department of Agriculture (USDA), National Science Foundation (NSF), Foundation for Food & Agricultural Research (FFAR), National Corn Growers Association (NCGA), Iowa Corn Growers Association (ICGA)
:::

