<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.5.57">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Presentations</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-bootstrap" data-mode="light">
<link href="site_libs/bootstrap/bootstrap-dark.min.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-bootstrap" data-mode="dark">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


<link rel="stylesheet" href="styles.css">
</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" role="menu" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="./index.html"> 
<span class="menu-text">Home üåΩ</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./program_schedule.html"> 
<span class="menu-text">Schedule</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link active" href="./generated_talks.html" aria-current="page"> 
<span class="menu-text">Talk Abstracts</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./generated_keynote_list.html"> 
<span class="menu-text">Keynote Abstracts</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./generated_lightning_list.html"> 
<span class="menu-text">Lightning Abstracts</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./program_poster_list.html"> 
<span class="menu-text">Poster Lookup</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./generated_poster_list.html"> 
<span class="menu-text">Poster Abstracts</span></a>
  </li>  
</ul>
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item">
    <a class="nav-link" href="./program_map.html"> 
<span class="menu-text">Map</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./program_general_information.html"> 
<span class="menu-text">General Information</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./program_awards.html"> 
<span class="menu-text">Awards</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./program_authors_attendees.html"> 
<span class="menu-text">Authors &amp; Attendees</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./program_sponsors.html"> 
<span class="menu-text">Sponsors</span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
            <div class="quarto-navbar-tools">
  <a href="" class="quarto-color-scheme-toggle quarto-navigation-tool  px-1" onclick="window.quartoToggleColorScheme(); return false;" title="Toggle dark mode"><i class="bi"></i></a>
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar zindex-bottom">
        
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Presentations</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<section id="t1-vgt1-as-enhancer-of-zmrap2.7-impacts-flowering-time-and-gene-regulatory-networks-involved-in-jasmonate-signaling-in-maize" class="level2">
<h2 class="anchored" data-anchor-id="t1-vgt1-as-enhancer-of-zmrap2.7-impacts-flowering-time-and-gene-regulatory-networks-involved-in-jasmonate-signaling-in-maize">T1: Vgt1 as enhancer of ZmRap2.7 impacts flowering time and gene regulatory networks involved in jasmonate signaling in maize</h2>
<p><font color="#339966">Cell and Developmental Biology </font> <a href="mailto:m.e.stam@uva.nl">Maike Stam</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-1-contents" aria-controls="callout-1" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-1" class="callout-1-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:johan.zicola@uni-goettingen.de">Zicola, Johan</a><sup>1 2</sup><br><a href="mailto:blaise.weber@gmail.com">Weber, Blaise</a><sup>3</sup><br><a href="mailto:xiaoyutu@sjtu.edu.cn">Tu, Xiaoyu</a><sup>4</sup><br><a href="mailto:R.Bader@uva.nl">Bader, Rechien</a><sup>3</sup><br><a href="mailto:dzisis1986@gmail.com">Zisis, Dimitrios</a><sup>5</sup><br><a href="mailto:stijn.aesaert@psb.vib-ugent.be">Aesaert, Stijn</a><sup>6</sup><br><a href="mailto:silvio.salvi@unibo.it">Salvi, Silvio</a><sup>7</sup><br><a href="mailto:pkra@igr.poznan.pl">Krajewski, Pawel</a><sup>5</sup><br><a href="mailto:mieke.vanlijsebettens@telenet.be">van Lijsebettens, Mieke</a><sup>6</sup><br><a href="mailto:lichuanshun@sjtu.edu.cn">Li, Chuanshun</a><sup>4</sup><br><a href="mailto:lymh96@sjtu.edu.cn">Li, Yangmeihui</a><sup>4</sup><br><a href="mailto:silin.zhong@cuhk.edu.hk">Zhong, Silin</a><sup>8</sup><br><a href="mailto:stefan.scholten@uni-goettingen.de">Scholten, Stefan</a><sup>2</sup><br><a href="mailto:turck@mpipz.mpg.de">Turck, Franziska</a><sup>1</sup><br><a href="mailto:m.e.stam@uva.nl">Stam, Maike</a><sup>3</sup><br></p>
<p><sup>1</sup>Max Planck Institute for Plant Breeding, Carl-von-Linn√© 10, 50829 Cologne, Germany<br><sup>2</sup>Georg-August-Universit√§t G√∂ttingen, Von-Siebold-Str. 8, 37075 G√∂ttingen, Germany<br><sup>3</sup>University of Amsterdam, Science Park 904, 1098 XH Amsterdam, The Netherlands<br><sup>4</sup>Shanghai Jiao Tong University, Shanghai, 200240, China<br><sup>5</sup>Institute of Plant Genetics, Polish Academy of Science, Strzeszy≈Ñska 34, 60-479 Pozna≈Ñ, Poland<br><sup>6</sup>VIB-UGent Center for Plant Systems Biology, Technologiepark 9279052 Gent, Belgium<br><sup>7</sup>DISTAL, University of Bologna,Viale Fanin 44, 40127, Bologna, Italy<br><sup>8</sup>School of Life Sciences, The Chinese University of Hong Kong, Hong Kong, China<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-2-contents" aria-controls="callout-2" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-2" class="callout-2-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>The identification and characterization of&nbsp;cis-regulatory DNA sequences and how they coordinate responses to developmental and environmental cues is of paramount importance to plant biology1. Although thousands of candidate&nbsp;cis-regulatory modules (CRMs) have been identified in maize, few&nbsp;of these&nbsp;have been well characterized1-3. We are studying&nbsp;the function of the&nbsp;Zea mays&nbsp;candidate enhancer&nbsp;Vegetative to generative1&nbsp;(Vgt1), a predicted regulatory element located about 70 kb upstream of the floral repressor gene&nbsp;ZmRap2.74. Consistent with a function as enhancer of&nbsp;ZmRap2.7,&nbsp;Vgt1&nbsp;contains an accessible chromatin region.&nbsp;Transgenic lines containing an inverted repeat that induces DNA methylation at&nbsp;Vgt1&nbsp;through&nbsp;RNA-directed DNA methylation (RdDM)&nbsp;show early flowering and accelerated growth rate during early developmental stages. DNA methylation of&nbsp;Vgt1&nbsp;is associated with downregulated expression of the AP2-like floral repressor&nbsp;ZmRap2.7&nbsp;in specific leaf tissues at early developmental stages. In line with&nbsp;Vgt1&nbsp;regulating&nbsp;ZmRap2.7, chromosome conformation capture data shows that&nbsp;Vgt1&nbsp;physically interacts with the&nbsp;ZmRap2.7&nbsp;TSS region. Finally,&nbsp;chromatin immunoprecipitation of transiently expressed&nbsp;ZmRap2.7&nbsp;in protoplasts indicates that the ZmRAP2.7 transcription factor binds at the promoter of several hundreds of genes, including a significant proportion of genes that are differentially expressed between lines with and without DNA methylation at&nbsp;Vgt1.&nbsp;Altogether, we show that&nbsp;ZmRap2.7&nbsp;is transcriptionally controlled by&nbsp;Vgt1&nbsp;and is involved in flowering time and other biological pathways, such as jasmonate signaling.1.&nbsp;&nbsp;&nbsp;Schmitz, Grotewold, Stam (2022) The Plant Cell 34:718. doi.org/10.1093/plcell/koab281.&nbsp;2.&nbsp;&nbsp;&nbsp;Oka et al.&nbsp;(2017)&nbsp;Genome Biology 18:137.&nbsp;doi: 10.1186/s13059-017-1273-4.&nbsp;3.&nbsp;&nbsp;&nbsp;Ricci et al (2019) Nat Plants 5: 1237. doi: 10.1038/s41477-019-0547-0. 4.&nbsp;&nbsp;&nbsp;Salvi et al.&nbsp;(2007) PNAS 104:11376. doi: 10.1073/pnas.0704145104.</p>
</div>
</div>
</div>
</section>
<section id="t2-decoding-a-complex-distal-non-coding-qtl-at-teosinte-branched-1" class="level2">
<h2 class="anchored" data-anchor-id="t2-decoding-a-complex-distal-non-coding-qtl-at-teosinte-branched-1">T2: Decoding a complex distal non-coding QTL at TEOSINTE BRANCHED 1</h2>
<p><font color="#336699">Biochemical and Molecular Genetics </font> <a href="mailto:ankush.sangra@uga.edu">Ankush Sangra</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-3-contents" aria-controls="callout-3" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-3" class="callout-3-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Sangra, Ankush<sup>1</sup><br>Chen, Zongliang<sup>2</sup><br>Marand, Alexandre P.<sup>1</sup><br>Mendieta, John P.<sup>1</sup><br>Minow, Mark A.A.<sup>1</sup><br>Clause, Haley N.<sup>1</sup><br>Studer, Anthony J.<sup>3</sup><br>Gallavotti, Andrea<sup>2</sup><br>Schmitz, Robert J.<sup>1</sup><br></p>
<p><sup>1</sup>Department of Genetics, University of Georgia Athens, USA<br><sup>2</sup>Waksman Institute of Microbiology, Rutgers University, Piscataway, NJ<br><sup>3</sup>Department of Crop Sciences, University of Illinois, Urbana, IL, USA<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-4-contents" aria-controls="callout-4" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-4" class="callout-4-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>During domestication from teosinte, maize branching and inflorescence architecture changed drastically, mainly due to increased TEOSINTE BRANCHED 1 (TB1) axillary bud expression. Elevated TB1 expression is due to domestication altering a distal control region (CR) of ~11kb. This domesticated CR has four accessible chromatin regions (ACRs), three of which are consistent with enhancer activity, as they co-occur with TB1 expression in developing female inflorescence. The final ACR appears consistent with silencer activity, as it is always accessible, even when the TB1 locus is repressed by H3K27me3 deposition. In contrast, most teosinte contain only three ACRs, as a Hopscotch transposon bifurcates one ancestral ACR in domesticated maize. Here, we functionally dissect the TB1 CR via CRISPR-Cas9 deletion of each CR ACR, Hopscotch, and the entire CR. Entire CR deletion phenocopy TB1 loss of function, confirming that the CR cumulatively enhances TB1 expression. Deletion of the two ACRs flanking Hopscotch caused weak tb1 branching phenotypes, suggesting additive enhancer action; indeed, these deletions caused reduced axillary bud TB1 expression, but no expression change in the leaf, where TB1 is normally repressed. Deletion of the putative silencer ACR caused no overt phenotypic change, yet preliminary results indicate that it changes the chromatin accessibility of the TB1 promoter. Surprisingly, chromatin at Hopscotch is not accessible and Hopscotch deletion produced no phenotypic changes ‚Äì this, along with the presence of this Hopscotch insertion in select teosinte accessions, suggest changes other than the Hopscotch insertion caused the domestication increase in maize apical dominance. Future exploration will characterize the molecular changes accompanying our ACR deletions in the TB1 CR and reveal more about the genetics that underpins TB1&nbsp;expression control.</p>
</div>
</div>
</div>
</section>
<section id="t3-cross-species-modeling-of-plant-genomes-at-single-nucleotide-resolution-using-a-pre-trained-dna-language-model" class="level2">
<h2 class="anchored" data-anchor-id="t3-cross-species-modeling-of-plant-genomes-at-single-nucleotide-resolution-using-a-pre-trained-dna-language-model">T3: Cross-species modeling of plant genomes at single nucleotide resolution using a pre-trained DNA language model</h2>
<p><font color="#663399">Computational and Large-Scale Biology </font> <a href="mailto:jz963@cornell.edu">Jingjing Zhai</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-5-contents" aria-controls="callout-5" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-5" class="callout-5-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:jz963@cornell.edu">Zhai, Jingjing</a><sup>1</sup><br><a href="mailto:akg87@cornell.edu">Gokaslan, Aaron</a><sup>2</sup><br><a href="mailto:yzs2@cornell.edu">Schiff, Yair</a><sup>2</sup><br><a href="mailto:ahb232@cornell.edu">Berthel, Ana</a><sup>1</sup><br><a href="mailto:zl843@cornell.edu">Liu, Zong-Yan</a><sup>3</sup><br><a href="mailto:wl748@cornell.edu">Lai, Wei-Yun</a><sup>1</sup><br><a href="mailto:zrm22@cornell.edu">Miller, Zachary</a><sup>1</sup><br><a href="mailto:armin.scheben@gmail.com">Scheben, Armin</a><sup>5</sup><br><a href="mailto:mcs368@cornell.edu">Stitzer, Michelle</a><sup>1</sup><br><a href="mailto:mcr72@cornell.edu">Romay, Cinta</a><sup>1</sup><br><a href="mailto:ed.buckler@usda.gov">Buckler, Edward</a><sup>1 3 4</sup><br><a href="mailto:vk379@cornell.edu">Kuleshov, Volodymyr</a><sup>2</sup><br></p>
<p><sup>1</sup>Institute for Genomic Diversity, Cornell University, Ithaca, NY USA 14853<br><sup>2</sup>Department of Computer Science, Cornell University, Ithaca, NY, USA 14853<br><sup>3</sup>Section of Plant Breeding and Genetics, Cornell University, Ithaca, NY USA 14853<br><sup>4</sup>USDA-ARS; Ithaca, NY, USA 14853<br><sup>5</sup>Simons Center for Quantitative Biology, Cold Spring Harbor Laboratory, 1 Bungtown Road, Cold Spring Harbor, NY USA 11724<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-6-contents" aria-controls="callout-6" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-6" class="callout-6-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Interpreting function and fitness effects in diverse plant genomes requires transferable machine learning models. Language models (LMs) pre-trained on large-scale unlabeled biological sequences can learn evolutionary conservation. With fine-tuning on limited labeled data, these models offer superior cross-species prediction compared to supervised approaches. Here, we introduce PlantCaduceus, a DNA LM built on the Caduceus and Mamba architectures, pre-trained on 16 angiosperm genomes spanning 160 million years of divergence. Using a masked language modeling framework, where each nucleotide is treated as a ‚Äúword,‚Äù PlantCaduceus captures evolutionarily constrained sequence patterns and achieves robust predictive performance with minimal labeled data. When fine-tuned on Arabidopsis for four key annotation tasks, PlantCaduceus demonstrates high transferability across divergent plant species. These tasks include translation initiation and termination site (TIS/TTS) prediction and splice donor/acceptor site identification, where it outperforms existing DNA LMs by up to 7.23-fold. For example, in maize TIS prediction, it achieves a PRAUC of 0.815, far exceeding the best baseline model‚Äôs 0.089. In identifying deleterious mutations, PlantCaduceus matches the performance of state-of-the-art protein LMs and surpasses phylogenetic approaches (PhyloP/phastCons) by up to threefold without fine-tuning. Additionally, it successfully identifies known causal variants in both Arabidopsis and maize. Together, these findings highlight PlantCaduceus as a versatile and powerful foundation model with broad utility for plant genomic annotation and crop improvement.</p>
</div>
</div>
</div>
</section>
<section id="t4-quantitative-genetics-of-leaf-vascular-density-in-maize" class="level2">
<h2 class="anchored" data-anchor-id="t4-quantitative-genetics-of-leaf-vascular-density-in-maize">T4: Quantitative genetics of leaf vascular density in maize</h2>
<p><font color="#339966">Cell and Developmental Biology </font> <a href="mailto:ruggiedi@oregonstate.edu">Diana Ruggiero</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-7-contents" aria-controls="callout-7" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-7" class="callout-7-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:ruggiedi@oregonstate.edu">Ruggiero, Diana</a><sup>1</sup><br><a href="mailto:georgechuck@berkeley.edu">Chuck, George</a><sup>2</sup><br><a href="mailto:leiboffs@oregonstate.edu">Leiboff, Samuel</a><sup>1</sup><br></p>
<p><sup>1</sup>Department of Botany and Plant Pathology, Oregon State University, Corvallis, OR 97331, USA.<br><sup>2</sup>Department of Plant and Microbial Biology, UC Berkeley, Berkeley, CA 94720, USA.<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-8-contents" aria-controls="callout-8" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-8" class="callout-8-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>C4 photosynthesis requires high vascular density to shuttle carbon from mesophyll to bundle sheath cells in adjacent veins. Maize leaves achieve high vascular density by producing several vein subtypes (lateral, intermediate, small, and transverse), defined by their sequence of development and spatial configuration. To identify genes influencing vascular density across vein subtypes, we phenotyped the Wisconsin Diversity Panel (WiDiv) and conducted a genome-wide association study (GWAS). Over three field seasons, we collected 6000+ leaf samples from 760 WiDiv inbred lines and built a deep-learning phenotyping system that estimates subtype-specific vein density in images of cleared leaf tissues. The system uses two U-Net convolutional neural networks (CNN) for semantic, pixel-by-pixel image segmentation: one model performs vein subtype classification, while the second model segments leaf images into sheath, auricle, and blade, allowing for compartment specific phenotyping. We used activation mapping and ‚ÄòDeepDream‚Äô-style feature visualization to show how the models interpret the leaf images. These models determined that veins make up, on average, 61% of the photosynthetically active blade and 51% of the non-photosynthetic sheath. Using publicly available field data, we found that several of our vascular phenotypes have modest, significant correlations with agriculturally relevant traits, including meristem size and flowering time. We additionally uncovered ‚Äòbundle sheath fusions‚Äô across many lines in the WiDiv, where ectopic bundle sheath cells develop in-between vascular bundles instead of mesophyll cells, violating the expected cellular spacing rules of C4 Kranz anatomy. Using our GWAS pipeline, we identified 160 significant loci associated with quantitative variation from our compartment and subtype-specific vascular phenotypes. To evaluate gene candidates, we are using hybridization chain reaction (HCR) RNA in situ to localize transcript expression during leaf development. Preliminary physiological data shows correlations between vascular traits and stomatal conductance. Future work will determine the physiological consequences&nbsp; and developmental origins of these traits.</p>
</div>
</div>
</div>
</section>
<section id="t5-integrating-proximal-sensing-modalities-for-enhanced-prediction-of-agronomically-important-crop-traits" class="level2">
<h2 class="anchored" data-anchor-id="t5-integrating-proximal-sensing-modalities-for-enhanced-prediction-of-agronomically-important-crop-traits">T5: Integrating proximal sensing modalities for enhanced prediction of agronomically important crop traits</h2>
<p><font color="#996633">Quantitative Genetics &amp; Breeding</font> <a href="mailto:eef52@cornell.edu">Erin Farmer</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-9-contents" aria-controls="callout-9" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-9" class="callout-9-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:eef52@cornell.edu">Farmer, Erin E.</a><sup>1</sup><br><a href="mailto:pm568@cornell.edu">Michael, Peter</a><sup>2</sup><br><a href="mailto:zg45@cornell.edu">Gu, Zeqi</a><sup>5</sup><br><a href="mailto:ruyu.yan@princeton.edu">Yan, Ruyu</a><sup>2</sup><br><a href="mailto:lw454@cornell.edu">Wickes-Do, Liam</a><sup>1</sup><br><a href="mailto:nkl3@cornell.edu">Lepak, Nick</a><sup>3</sup><br><a href="mailto:mcr72@cornell.edu">Romay, Cinta</a><sup>4</sup><br><a href="mailto:esb33@cornell.edu">Buckler, Edward S.</a><sup>1 3 4</sup><br><a href="mailto:ys776@cornell.edu">Sun, Ying</a><sup>6</sup><br><a href="mailto:krr73@cornell.edu">Robbins, Kelly</a><sup>1</sup><br><a href="mailto:jlgage@ncsu.edu">Gage, Joseph L.</a><sup>7</sup><br><a href="mailto:abedavis@cornell.edu">Davis, Abe</a><sup>2</sup><br><a href="mailto:mag87@cornell.edu">Gore, Michael A.</a><sup>1</sup><br></p>
<p><sup>1</sup>Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, New York, USA 14853<br><sup>2</sup>Department of Computer Science, Cornell University, Ithaca, New York, USA 14853<br><sup>3</sup>United States Department of Agriculture-Agricultural Research Service, Robert W. Holley Center for Agriculture and Health, Ithaca, New York, USA 14853<br><sup>4</sup>Institute for Genomic Diversity, Cornell University, Ithaca, New York, USA 14853<br><sup>5</sup>Cornell Tech, New York, New York, USA 10044<br><sup>6</sup>Soil and Crop Sciences Section, School of Integrative Plant Science, Cornell University, Ithaca, NY, USA 14853<br><sup>7</sup>Department of Crop and Soil Sciences, North Carolina State University, Raleigh, North Carolina, USA 27695<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-10-contents" aria-controls="callout-10" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-10" class="callout-10-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Proximal sensing technologies have enabled the collection of vast amounts of phenomic data for characterization of crop traits. However, data streams collected across sensors and platforms are predominately utilized separately due to heterogeneity in their structural, spatial, and spectral information. We aim to leverage advances in artificial intelligence to characterize plant form and health across temporal scales through integration of high-dimensional, multi-modal data. We collected over 75,000 multispectral images (MSIs) via unoccupied aerial vehicles (UAVs) and over 35,000 LiDAR scans via unoccupied ground vehicles (UGVs) for maize hybrids from the Genomes to Fields project in Aurora, NY from 2020 through 2024. Autoencoders, which are unsupervised, deep learning models, were implemented to extract latent features from each data type, producing reduced representations which contained biologically relevant information, with heritabilities up to ~0.9. MSI and LiDAR latent features were integrated at the plot-level across all time points, and Bayesian regression was used to predict manually measured phenotypes. These predictions were compared against 49 vegetation indices (VIs) which had accuracies ranging from ~0.26 for stalk lodging to ~0.85 for days to anthesis. Integrated latent features outperformed VIs for all phenotypes except for flowering time traits and grain moisture, ranging from a decrease in accuracy of ~8.8% for grain moisture to an increase of ~19.0% for ear height. Across all traits, integrated latent features increased the prediction accuracy by an average of ~4.6%. Notably, integrated latent features also provided a ~5.1% and ~20.8% increase in accuracy over the individual use of MSI and LiDAR latent features, respectively. We show that latent phenotyping circumvents manual curation of features and allows integration across modalities, improving characterization of key crop traits and further facilitating the deployment of proximal sensors for use in precision agriculture and breeding programs.</p>
</div>
</div>
</div>
</section>
<section id="t6-crowdsourcing-phenotype-prediction-results-from-the-2024-g2f-prediction-competition." class="level2">
<h2 class="anchored" data-anchor-id="t6-crowdsourcing-phenotype-prediction-results-from-the-2024-g2f-prediction-competition.">T6: Crowdsourcing phenotype prediction: Results from the 2024 G2F prediction competition.</h2>
<p><font color="#996633">Quantitative Genetics &amp; Breeding</font> <a href="mailto:jacob.washburn@usda.gov">Jacob Washburn</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-11-contents" aria-controls="callout-11" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-11" class="callout-11-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Winner, Competition<sup>1</sup><br>Washburn, Jacob D.<sup>2</sup><br>Chen, Qiuyue<sup>3</sup><br>de Leon, Natalia<sup>3</sup><br>Gage, Joseph<sup>4 5</sup><br>Holland, James<sup>6</sup><br>Lima, Dayane<sup>3</sup><br>Murray, Seth<sup>7</sup><br>Romay, Cinta<sup>8</sup><br>Taylor, Kerry<sup>9</sup><br>Xavier, Alencar<sup>10 11</sup><br></p>
<p><sup>1</sup>Competition Winner Affiliation<br><sup>2</sup>USDA-ARS, Plant Genetics Research Unit, University of Missouri, Columbia, MO 65211<br><sup>3</sup>Department of Plant and Agroecosystem Sciences, University of Wisconsin, Madison, WI 53706<br><sup>4</sup>Department of Crop and Soil Sciences, North Carolina State University, Raleigh, NC 27695<br><sup>5</sup>NC Plant Sciences Initiative, North Carolina State University, Raleigh, NC, 27606<br><sup>6</sup>USDA-ARS, Plant Science Research Unit, North Carolina State University, Raleigh, NC 27606<br><sup>7</sup>Department of Soil &amp; Crop Sciences, Texas A&amp;M University, College Station, TX 77843<br><sup>8</sup>Institute for Genomic Diversity, Cornell University, Ithaca, NY 14850<br><sup>9</sup>Iowa Corn Promotion Board, Johnston, IA 50131<br><sup>10</sup>Corteva Agrisciences, 8305 NW 62nd Ave, Johnston, IA, 50131, USA<br><sup>11</sup>Department of Agronomy, Purdue University, 915 Mitch Daniels Blvd, West Lafayette, IN, 47907, USA<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-12-contents" aria-controls="callout-12" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-12" class="callout-12-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Predicting yield in new environments and new genetics is notoriously difficult, but accurate predictions have important ramifications for breeding, management, sustainability, and the environment. The maize Genomes to Fields (G2F) genotype by environment (GxE) project was conceived more than a decade ago as a multi-state cooperative to test hypotheses and enhance prediction of maize yield, adaptation, and environmental responses. To date the project has phenotyped around 190,000 unique plots, around 6,100 hybrids, and more than 300 environments. This year the GxE project held its second international yield prediction competition using the complete G2F GxE project dataset to date! The goal was for teams to predict the entirely unseen data collected in summer 2024. The competition attracted over 370 registrants from across the world with competitors from academia, government, non-profit, for profit, and private individuals. This presentation will announce the winning competitors to the community after which they will present the strategies, and finer details that enabled their model to outperform all others.</p>
</div>
</div>
</div>
</section>
<section id="t7-ensuring-the-future-of-maize-a-call-for-collaborative-action" class="level2">
<h2 class="anchored" data-anchor-id="t7-ensuring-the-future-of-maize-a-call-for-collaborative-action">T7: Ensuring the future of maize: A call for collaborative action</h2>
<p><font color="#993366">Education &amp; Outreach</font> <a href="mailto:vivian.bernau@usda.gov">Vivian Bernau</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-13-contents" aria-controls="callout-13" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-13" class="callout-13-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:vivian.bernau@usda.gov">Bernau, Vivian</a><sup>1</sup><br><a href="mailto:mark.millard@usda.gov">Millard, Mark</a><sup>1</sup><br><a href="mailto:adam.mahan@usda.gov">Mahan, Adam</a><sup>1</sup><br></p>
<p><sup>1</sup>USDA/ARS Plant Introduction Research Unit and Iowa State University College of Agriculture and Life Sciences, Ames, Iowa<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-14-contents" aria-controls="callout-14" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-14" class="callout-14-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>The USDA National Plant Germplasm System includes a collection of more than 20,000 accessions of cultivated temperate- and tropical-adapted maize and wild relatives from around the world. This collection is distributed from the North Central Regional Plant Introduction Station (NCRPIS) in Ames, Iowa, where it is conserved at 4¬∞C, and backed-up at the National Laboratory for Genetic Resources Preservation in Fort Collins, Colorado. The maize collection continues to grow each year, with the addition of materials with expiring plant variety protection and other resources important to the maize community. Additionally, the collection continues to age; more than half of the collection‚Äôs distributable seed lots are now more than 30 years old. Funding levels for maintenance and distribution have lagged behind the demand and replenishment needs of the maize collection for some time. In order to continue conserving diversity for our cultural heritage and to address current and future challenges, we seek 1) partnerships to regenerate and characterize the maize germplasm collection and 2) feedback to aid in the prioritization of our efforts.</p>
</div>
</div>
</div>
</section>
<section id="t8-translational-and-proteomic-analysis-of-cold-stressed-maize-reveals-ribosomal-protein-families-involved-in-cold-response-and-tolerance" class="level2">
<h2 class="anchored" data-anchor-id="t8-translational-and-proteomic-analysis-of-cold-stressed-maize-reveals-ribosomal-protein-families-involved-in-cold-response-and-tolerance">T8: Translational and proteomic analysis of cold-stressed maize reveals ribosomal protein families involved in cold response and tolerance</h2>
<p><font color="#336699">Biochemical and Molecular Genetics </font> <a href="mailto:vcp8@cornell.edu">Veronica C Perez</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-15-contents" aria-controls="callout-15" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-15" class="callout-15-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:vcp8@cornell.edu">Perez, Veronica C</a><sup>1</sup><br><a href="mailto:jh299@cornell.edu">Hua, Jian</a><sup>1</sup><br></p>
<p><sup>1</sup>Plant Biology Section, School of Integrative Plant Science, Cornell University, Ithaca, New York, USA<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-16-contents" aria-controls="callout-16" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-16" class="callout-16-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>At the onset of stress, plants undergo rapid translational reprogramming to promote the translation of response genes and minimize damage, and with extended stress periods may alter ribosomal composition to maximize response gene expression and restore overall translational efficiency and growth. In maize, exploration of this translational and ribosomal reprogramming has been sparse, especially in response to abiotic stressors such as cold. To identify genes including ribosomal protein (RP) paralogs affected at the translational level in response to cold, RNA-Seq, Ribo-Seq and MS/MS analyses were conducted on seedlings following 24hr cold stress. RNA analyses identified differentially expressed genes within total and ribosome-bound, translationally active RNA, showing both downregulation of translation and ribosome biogenesis related terms in actively translated RNA and upregulation of specific RP paralogs at the translational level. MS/MS analyses of ribosome-enriched samples identified RP families and individual paralogs that exhibited differential protein accumulation or phosphorylation in response to cold. Characterization of several cold tolerance candidate RP genes was conducted in Arabidopsis with closest homologs to determine the cold response function of these RPs. Preliminary data shows that knocking out of paralogs belonging to the uL18 and uL10 families, which showed cold responsive phosphorylation and translational induction in maize respectively, led to altered freezing tolerance and cold growth phenotypes in Arabidopsis. Together, these analyses identified several candidate RP genes that may play a role in maize or plant cold tolerance and provides a dataset for identification of other translationally regulated and cold responsive genes.</p>
</div>
</div>
</div>
</section>
<section id="t9-introgression-of-a-mexican-highland-chromosomal-inversion-into-temperate-maize-accelerates-flowering-promotes-growth-and-modulates-a-cell-proliferation-gene-network." class="level2">
<h2 class="anchored" data-anchor-id="t9-introgression-of-a-mexican-highland-chromosomal-inversion-into-temperate-maize-accelerates-flowering-promotes-growth-and-modulates-a-cell-proliferation-gene-network.">T9: Introgression of a Mexican highland chromosomal inversion into temperate maize accelerates flowering, promotes growth, and modulates a cell proliferation gene network.</h2>
<p><font color="#336699">Biochemical and Molecular Genetics </font> <a href="mailto:frodrig4@ncsu.edu">Fausto Rodr√≠guez-Zapata</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-17-contents" aria-controls="callout-17" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-17" class="callout-17-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:frodrig4@ncsu.edu">Rodr√≠guez-Zapata, Fausto</a><sup>1 2</sup><br><a href="mailto:ntanduk@ncsu.edu">Tandukar, Nirwan</a><sup>1 2</sup><br><a href="mailto:acbarne5@ncsu.edu">Barnes, Allison</a><sup>1</sup><br><a href="mailto:rstokes@ncsu.edu">Stokes, Ruthie</a><sup>1</sup><br><a href="mailto:jaaragon@ncsu.edu">Arag√≥n-Raygoza, Alejandro</a><sup>1</sup><br><a href="mailto:mul826@psu.edu">Li, Meng</a><sup>3</sup><br><a href="mailto:sjp6181@psu.edu">P√©rez-Lim√≥n, Sergio</a><sup>3</sup><br><a href="mailto:mpm5587@psu.edu">Perryman, Melanie</a><sup>3</sup><br><a href="mailto:map25@cornell.edu">Pi√±eros, Miguel A.</a><sup>4</sup><br><a href="mailto:jjstrabl@ncsu.edu">Strable, Josh</a><sup>1 2</sup><br><a href="mailto:deruncie@ucdavis.edu">Runcie, Daniel</a><sup>5</sup><br><a href="mailto:rjs6686@psu.edu">Sawers, Ruairidh J.</a><sup>3</sup><br><a href="mailto:rrellan@ncsu.edu">Rell√°n-√Ålvarez, Rub√©n</a><sup>1 2</sup><br></p>
<p><sup>1</sup>Molecular and Structural Biochemistry, North Carolina State University, Raleigh, North Carolina, USA<br><sup>2</sup>Genetics and Genomics, North Carolina State University, Raleigh, North Carolina, USA<br><sup>3</sup>Department of Plant Science, The Pennsylvania State University, University Park, PA, USA<br><sup>4</sup>Robert W. Holley Center for Agriculture and Health, USDA-ARS, Ithaca, NY, USA<br><sup>5</sup>Department of Plant Sciences, University of California, Davis, Davis, CA, USA<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-18-contents" aria-controls="callout-18" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-18" class="callout-18-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Inv4m is a chromosomal inversion prevalent in traditional maize varieties adapted to the cold and often phosphorus-deficient Mexican highlands. Field trials throughout Mexico have shown that, when grown at high elevations, plants carrying the inversion flower faster and have greater yield than plants without it. Although growth chamber experiments indicate that Inv4m regulates the expression of photosynthesis-related genes in response to cold, we have yet to know the genes responsible for the adaptive effects of Inv4m in the field. To identify Inv4m-regulated genes that underlie enhanced development in the field, we bred B73-based Near Isogenic Lines (NILs) with either the inversion or the standard karyotype. We then grew these NILs in phosphorus-sufficient and deficient soils to test whether Inv4m contributes to local adaptation through enhanced phosphorus stress response. We measured plant reproductive and vegetative traits, phosphorus, lipids, and gene expression in the leaves. Plants showed classical responses to phosphorus starvation, including decreased phosphorus and biomass accumulation, delayed flowering, and a switch from phospholipid to glycolipid production. Notably, Inv4m plants flowered earlier and grew taller regardless of phosphorus availability. While increased leaf age and phosphorus deficiency resulted in genome-wide expression changes, Inv4m‚Äôs effects were predominantly confined to genes within the inversion. Our analyses suggest that Inv4m introgression modulates a trans-coexpression network enriched in cell proliferation and flower development genes, which includes DNA replication fork genes (pcna2, mcm5), histone demethylases (jmj2, jmj21), and the FT florigen homolog zcn26. By cross-referencing with a list of candidates from the literature, we found other Inv4m-regulated genes associated with flowering time and plant height. In a complementary growth chamber experiment, Inv4m plants showed longer shoot apical meristems than controls, supporting its effect on organ development. These findings provide insights into Inv4m‚Äôs role in highland adaptation through the coordinated expression of a developmental gene network.</p>
</div>
</div>
</div>
</section>
<section id="t10-integration-of-phenomic-proteomic-and-genomic-data-into-a-multi-scale-network-unravels-missing-heritability-for-maize-response-to-water-deficit" class="level2">
<h2 class="anchored" data-anchor-id="t10-integration-of-phenomic-proteomic-and-genomic-data-into-a-multi-scale-network-unravels-missing-heritability-for-maize-response-to-water-deficit">T10: Integration of phenomic, proteomic, and genomic data into a multi-scale network unravels missing heritability for maize response to water deficit</h2>
<p><font color="#663399">Computational and Large-Scale Biology </font> <a href="mailto:marie-laure.martin@inrae.fr">Marie-Laure Martin</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-19-contents" aria-controls="callout-19" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-19" class="callout-19-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:djabaliyacine@yahoo.fr">Djabali, Yacine</a><sup>1 2 4</sup><br><a href="mailto:renaud.rincent@inrae.fr">Rincent, Renaud</a><sup>4</sup><br><a href="mailto:melisande.blein-nicolas@inrae.fr">Blein-Nicolas, M√©lisande</a><sup>4</sup><br><a href="mailto:marie-laure.martin@inrae.fr">Martin, Marie-Laure</a><sup>1 2 3</sup><br></p>
<p><sup>1</sup>Universit√© Paris-Saclay, CNRS, INRAE, Universit√© Evry, Institute of Plant Sciences Paris-Saclay (IPS2), 91190, Gif sur Yvette, France<br><sup>2</sup>Universit√© Paris Cit√©, Institute of Plant Sciences Paris-Saclay (IPS2), 91190, Gif sur Yvette, France<br><sup>3</sup>Universit√© Paris-Saclay, AgroParisTech, INRAE, UMR MIA Paris-Saclay, 91120, Palaiseau, France<br><sup>4</sup>Universit√© Paris-Saclay, INRAE, CNRS, AgroParisTech, GQE-Le Moulon, 91190, Gif-Sur-Yvette,<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-20-contents" aria-controls="callout-20" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-20" class="callout-20-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>The evolution of maize yields under water deficit conditions is of particular concern in the context of climate change and human population growth. However, the multiplicity and versatility of drought-response mechanisms make the design of new varieties a complex task that would greatly benefit from a better understanding of the genotype-phenotype relationship. &nbsp;The omnigenic model assumes that the entire genome contributes to complex traits, encouraging the consideration of small-effect SNPs that may explain missing heritability. Under such a hypothesis, we implemented here an innovative systems biology approach integrating the genetic determinism of molecular entities through the combination of genome-wide association studies and network inference. &nbsp;At each step of the integration process, a multi-environment mixed model was used to estimate the part of the genotype x water deficit interaction (GxW) variance captured by the genomic regions identified by our method.&nbsp;We applied our approach on a multi-omic dataset, including phenotypic, proteomic, and genomic data acquired from 254 maize hybrids grown under well-watered and water deficit conditions. &nbsp; Our results show that (i) QTLs&nbsp;underlying variations in&nbsp;protein abundance&nbsp;capture a part of the missing heritability of maize response to water stress response; (ii) there exists a synergy between the loci found in the two watering conditions and the loci associated with plasticity indices calculated from the two conditions; (iii) taking this synergy into account in our approach&nbsp;further&nbsp;increases&nbsp;the part of the GxW variance captured.We found about 400 new loci capturing 89.4%, 66.5%, and 77.5% of the GxW variance of biomass,&nbsp;water use efficiency, and stomatal conductance, respectively, which brings a gain up to 20 points in captured GxW variances. &nbsp;Hence&nbsp;our results show that multi-omics data integration can be an efficient way to capture missing heritability for complex phenotypic traits and identify new candidate genes related to drought response.</p>
</div>
</div>
</div>
</section>
<section id="t11-transcriptional-regulation-of-stress-adaptation-in-maize-identification-and-functional-annotation" class="level2">
<h2 class="anchored" data-anchor-id="t11-transcriptional-regulation-of-stress-adaptation-in-maize-identification-and-functional-annotation">T11: Transcriptional regulation of stress adaptation in maize: Identification and functional annotation</h2>
<p><font color="#663399">Computational and Large-Scale Biology </font> <a href="mailto:margaret.woodhouse@usda.gov">Maggie Woodhouse</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-21-contents" aria-controls="callout-21" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-21" class="callout-21-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:rita.hayford@usda.gov">HAYFORD, RITA K</a><sup>1</sup><br><a href="mailto:olivia.haley@usda.gov">Haley, Olivia C</a><sup>1</sup><br><a href="mailto:Laura.Tibbs-Cortes@usda.gov">Tibbs-Cortes, Laura</a><sup>1</sup><br><a href="mailto:ethy.cannon@usda.gov">Cannon, Ethalinda K</a><sup>1</sup><br><a href="mailto:john.portwood@usda.gov">Portwood II, John L</a><sup>1</sup><br><a href="mailto:jack.m.gardiner@gmail.com">Gardiner, Jack M</a><sup>2</sup><br><a href="mailto:carson.andorf@usda.gov">Andorf, Carson M</a><sup>1 3</sup><br><a href="mailto:margaret.woodhouse@usda.gov">Woodhouse, Margaret R</a><sup>1</sup><br></p>
<p><sup>1</sup>USDA-ARS, Corn Insects and Crop Genetics Research Unit, Ames, IA 50011, USA<br><sup>2</sup>Division of Animal Sciences, University of Missouri, Columbia, MO 65211, USA<br><sup>3</sup>Department of Computer Science, Iowa State University, Ames, IA 50011, USA<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-22-contents" aria-controls="callout-22" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-22" class="callout-22-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Maize (Zea mays ssp. mays) is an essential grain crop cultivated for food, animal feed, fiber, and biofuel; therefore, maize yield improvements are necessary for future food security. Despite the importance of maize, various biotic and abiotic stresses negatively impact maize growth and development. Plants possess complex gene regulatory mechanisms influencing their responses to different stress factors. Chromatin Immunoprecipitation and Sequencing (ChIP-Seq) is a powerful method to understand transcription factor binding sites that regulate stress-responsive genes and map histone modifications controlling gene expression during stress. However, ChIP-Seq studies in maize during stress are under-explored. Although combining RNA-Seq and ChIP-Seq would provide a deeper understanding of gene regulation, such integrated study in maize is limited. We previously mapped high-quality RNA-Seq reads from publicly available datasets on&nbsp; B73 experiments. In this work, we aimed to map ChIP-Seq stress data from experiments from B73 imposed with stress to understand better how stress-response genes are regulated at the chromatin level. By integrating differentially expressed genes with chromatin regulatory mechanisms, we identified potential target genes in maize that provide valuable insights into stress responses. Comparing differentially marked peaks from ChIP-Seq data with RNA-Seq results allowed us to pinpoint stress-responsive genes and transcription factors. Our study also highlights the need for high-quality ChIP-Seq data in maize under stress conditions. Additionally, it demonstrates the importance of combining ChIP-Seq and RNA-Seq approaches to uncover transcriptional regulatory mechanisms governing stress responses in maize and other plant species.</p>
</div>
</div>
</div>
</section>
<section id="t12-a-rootless1-knockdown-allele-affects-maize-nodal-root-development-increasing-rooting-depth-nitrogen-uptake-efficiency-and-grain-production-in-the-field" class="level2">
<h2 class="anchored" data-anchor-id="t12-a-rootless1-knockdown-allele-affects-maize-nodal-root-development-increasing-rooting-depth-nitrogen-uptake-efficiency-and-grain-production-in-the-field">T12: A Rootless1 knockdown allele affects maize nodal root development, increasing rooting depth, nitrogen uptake efficiency, and grain production in the field</h2>
<p><font color="#996633">Quantitative Genetics &amp; Breeding</font> <a href="mailto:aliu@danforthcenter.org">Alexander Liu</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-23-contents" aria-controls="callout-23" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-23" class="callout-23-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:aliu@danforthcenter.org">Liu, Alexander E</a><sup>1 2</sup><br><a href="mailto:dthiruppathi1@gmail.com">Thiruppathi, Dhineshkumar</a><sup>2</sup><br><a href="mailto:adamlbray@gmail.com">Bray, Adam L</a><sup>2</sup><br><a href="mailto:gbagnall@danforthcenter.org">Bagnall, George</a><sup>2</sup><br><a href="mailto:emorales@danforthcenter.org">Morales, Elisa</a><sup>2</sup><br><a href="mailto:clebow@danforthcenter.org">Lebow, Clara</a><sup>2</sup><br><a href="mailto:ctopp@danforthcenter.org">Topp, Christopher N</a><sup>2</sup><br></p>
<p><sup>1</sup>Washington University in Saint Louis, Saint Louis, MO, USA 63141<br><sup>2</sup>Donald Danforth Plant Science Center, Saint Louis, MO, USA 63132<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-24-contents" aria-controls="callout-24" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-24" class="callout-24-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Nodal roots dominate the maize root system and are critical for nutrient acquisition. Changing nodal rooting patterns could help improve root system function; for example, the ‚ÄúSteep, Cheap, and Deep‚Äù paradigm posits that high nodal root production just above the soil line, but no higher, could increase nitrogen capture. Rootless1, a classic maize mutant, produces very few nodal roots aboveground. We previously identified a large indel in the promoter of ZmRt1 that reduces expression which we hypothesize is responsible for the original Rootless1 phenotype. However, an Ac/Ds insertion allele of ZmRt1, named rt1-2, was observed to induce supernumerary nodal roots near the soil line before a precipitous decline at higher nodes in several maize backgrounds. We present a multi-year and multi-location analysis of changes to root system architecture due to the rt1-2 nodal rooting phenotype and functional effects on plant growth and nitrogen status. Nitrogen contrast field experiments were performed in 2022, 2023, and 2024 in Missouri and Colorado comparing the rt1-2 allele to the ZmRt1 wild type allele in conventional and nitrogen limited conditions. Root system architecture was analyzed from excavated root crowns, 1m soil cores, and minirhizotrons. X-ray tomography analysis of excavated root crowns showed significant differences in root crown development. Deep soil cores revealed that plants with the rt1-2 allele have increased root length deeper in the soil column while minirhizotron data collected over the growing season showed differences in root system growth over time. To measure the functional impact of the observed changes in root system architecture, aboveground measures including shoot biomass, shoot nitrogen, and grain production were collected. Plants with the rt1-2 allele had higher concentrations of shoot nitrogen, potentially leading to the observed increase in grain in both conventional and low nitrogen conditions when compared to the ZmRt1 wild type allele, suggesting a positive influence on root resource capture efficiency.</p>
</div>
</div>
</div>
</section>
<section id="t13-overexpression-of-pstol1-like-genes-increases-maize-root-surface-area-and-biomass-under-low-and-high-phosphorus-conditions" class="level2">
<h2 class="anchored" data-anchor-id="t13-overexpression-of-pstol1-like-genes-increases-maize-root-surface-area-and-biomass-under-low-and-high-phosphorus-conditions">T13: Overexpression of PSTOL1-like genes increases maize root surface area and biomass under low and high phosphorus conditions</h2>
<p><font color="#336699">Biochemical and Molecular Genetics </font> <a href="mailto:sylvia.sousa@embrapa.br">Sylvia Morais de Sousa Tinoco</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-25-contents" aria-controls="callout-25" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-25" class="callout-25-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:barbarafnegri@gmail.com">Negri, Barbara F.</a><sup>1</sup><br><a href="mailto:laianesm@gmail.com">Maciel, Laiane S.</a><sup>2</sup><br><a href="mailto:antoniomarcos.coelho@embrapa.br">Coelho, Antonio M.</a><sup>3</sup><br><a href="mailto:beatriz.barros@embrapa.br">Barros, Beatriz A.</a><sup>3</sup><br><a href="mailto:claudia.guimaraes@embrapa.br">Guimar√£es, Claudia T.</a><sup>1 2 3</sup><br><a href="mailto:jurandir.magalhaes@embrapa.br">Magalh√£es, Jurandir V.</a><sup>1 2 3</sup><br><a href="mailto:sylvia.sousa@embrapa.br">Tin√¥co, Sylvia M. S.</a><sup>1 3</sup><br></p>
<p><sup>1</sup>Federal University of S√£o Jo√£o del-Rei (UFSJ), S√£o Jo√£o del-Rei, MG, Brazil, 36301-158<br><sup>2</sup>Federal University of Minas Gerais (UFMG), Belo Horizonte, MG, Brazil, 31270-901<br><sup>3</sup>Embrapa Maize and Sorghum, Sete Lagoas, MG, Brazil, 35701-970<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-26-contents" aria-controls="callout-26" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-26" class="callout-26-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Low phosphorus (P) availability in soil is a significant limitation for crop production in tropical regions. The PHOSPHORUS-STARVATION TOLERANCE1 (OsPSTOL1) protein kinase enhances root surface area, P acquisition, and grain yield in rice under P-deficient conditions. Homologs of OsPSTOL1 in sorghum were identified through association mapping in two sorghum panels phenotyped for P uptake, root system morphology, and architecture in hydroponic systems, as well as grain yield and biomass accumulation under low-P conditions in Brazil and Mali. In maize and sorghum, candidate genes were co-localized with quantitative trait loci (QTL) associated with root morphology, dry weight, and grain yield under low P. To validate the function of these genes, the rice OsPSTOL1 (as a positive control) and its maize (ZmPSTOL3.06, ZmPSTOL8.02, and ZmPSTOL8.05_1) and sorghum (Sb07g002840, Sb03g031690, and Sb03g006765) homologs were cloned downstream of an ubiquitin promoter in the pMCG1005 vector, with the Bar gene serving as a selective marker. Genetic transformation of maize B104 embryos was performed using Agrobacterium tumefaciens. Homozygous transgenic events with a single copy of the transgene were selected, and those showing transgene overexpression were evaluated under low and high-P conditions. In a growth chamber, the events Sb07g002840, Sb03g006765, and ZmPSTOL3.06 showed greater root length and total fine root surface area compared to the negative control (B104 transformed with an empty vector), under both low and high P-conditions. The Sb03g006765 event exhibited higher root and shoot dry weight under both low and high P. ZmPSTOL8.02 presented a higher dry weight than the negative control only under low P, while Sb07g002840 had a higher shoot dry weight under high P. In the greenhouse, the Sb03g006765, ZmPSTOL8.02, and ZmPSTOL8.05_1 events showed increased shoot dry weight under low P, while OsPSTOL1, Sb07g002840, and Sb03g006765 exhibited higher shoot dry weight under high P. For root dry weight under low P, Sb03g006765, ZmPSTOL8.02, and ZmPSTOL8.05_1 were superior to the negative control. &nbsp;Under high P, all events except ZmPSTOL8.02 outperformed the negative control. Overexpression of the PSTOL1 homologs significantly improved vegetative growth and root surface area, demonstrating that these genes function similarly to OsPSTOL1 in rice.</p>
</div>
</div>
</div>
</section>
<section id="t14-delayed-divisions-and-cell-elongation-defects-influence-plant-growth-in-katanin-mutants" class="level2">
<h2 class="anchored" data-anchor-id="t14-delayed-divisions-and-cell-elongation-defects-influence-plant-growth-in-katanin-mutants">T14: Delayed divisions and cell elongation defects influence plant growth in katanin mutants</h2>
<p><font color="#339966">Cell and Developmental Biology </font> <a href="mailto:smart046@ucr.edu">Stephanie Martinez</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-27-contents" aria-controls="callout-27" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-27" class="callout-27-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:smart046@ucr.edu">Martinez, Stephanie E</a><sup>1</sup><br><a href="mailto:lindy.allsman@ucr.edu">Allsman, Lindy A</a><sup>1</sup><br><a href="mailto:iceba004@ucr.edu">Ceballos, Ian</a><sup>1</sup><br><a href="mailto:cirah006@ucr.edu">Irahola, Cassandra</a><sup>1</sup><br><a href="mailto:kin.lau@vai.org">Lau, Kin H</a><sup>2</sup><br><a href="mailto:ajwright@gmail.com">Wright, Amanda J</a><sup>3</sup><br><a href="mailto:cweil@nsf.gov">Weil, Clifford</a><sup>4 5</sup><br><a href="mailto:crasmu@ucr.edu">Rasmussen, Carolyn G</a><sup>1</sup><br></p>
<p><sup>1</sup>Department of Botany and Plant Sciences, Center for Plant Cell Biology, Institute for Integrative Genome Biology, University of California Riverside, Riverside, CA 92521, USA<br><sup>2</sup>Bioinformatics and Biostatistics Core, Van Andel Institute, Grand Rapids, MI 49503, USA<br><sup>3</sup>Unaffiliated, Lewisville, TX, USA<br><sup>4</sup>Purdue University, West Lafayette, IN 47907, USA<br><sup>5</sup>Division of Molecular and Cellular Biosciences, National Science Foundation, Alexandria, Virginia 22314, USA<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-28-contents" aria-controls="callout-28" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-28" class="callout-28-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Microtubule dynamics and organization influence cell shape and cell division plane orientation. One protein complex involved in this process is KATANIN, a microtubule severing AAA ATPase hexameric complex composed of catalytic p60 and regulatory WD-40-containing p80 subunits. In Zea mays, two genes encode KATANIN (p60), and mutants have been identified called discordia3a (dcd3a) and dcd3b. Live cell imaging showed reduced microtubule severing frequency in dcd3a-2 dcd3b double mutants compared to wild-type siblings. Similar to katanin mutants in other organisms, dcd3a-2 dcd3b mutants have decreased plant height. To determine if defects in cell elongation cause the reductions in leaf area, cell dimensions were measured. When compared to wild-type siblings, dcd3a-2 dcd3b mutants have smaller epidermal cells, as seen by significant reduction in cell area and length to width ratios, but these small cells are not the primary cause of reduced leaf area. Instead, the dcd3a dcd3b mutants have approximately four times fewer cells. To determine if cell division delays cause fewer cells, we measured cell division progression in dcd3a-2 dcd3b and wild-type siblings. There was no significant delay from metaphase until the completion of cytokinesis. However, a significant decrease in the proportion of cells in late G2 and prophase in dcd3a-2 dcd3b double mutant suggests delays in mitotic entry. Therefore, a combination of cell elongation defects and delayed mitotic entry generates small mutant plants.</p>
</div>
</div>
</div>
</section>
<section id="t15-the-epf-erecta-ligand-receptor-pairs-regulate-maize-shoot-and-inflorescence-architecture-in-coordination-with-clavata-pathway-in-maize" class="level2">
<h2 class="anchored" data-anchor-id="t15-the-epf-erecta-ligand-receptor-pairs-regulate-maize-shoot-and-inflorescence-architecture-in-coordination-with-clavata-pathway-in-maize">T15: The EPF-ERECTA ligand-receptor pairs regulate maize shoot and inflorescence architecture in coordination with CLAVATA pathway in maize</h2>
<p><font color="#339966">Cell and Developmental Biology </font> <a href="mailto:fxu@sdu.edu.cn">Fang Xu</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-29-contents" aria-controls="callout-29" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-29" class="callout-29-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Liu, Xiao<sup>1</sup><br><a href="mailto:jacksond@cshl.edu">Jackson, David</a><sup>2</sup><br><a href="mailto:fxu@sdu.edu.cn">Xu, Fang</a><sup>1</sup><br></p>
<p><sup>1</sup>Shandong University, Qingdao, Shandong, China, 266237<br><sup>2</sup>Cold Spring Harbor Laboratory, Cold Spring Harbor, NY, USA,11724<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-30-contents" aria-controls="callout-30" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-30" class="callout-30-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>In maize, several key yield-related traits are associated with meristem activity, regulated by CLE peptide signals recognized by receptor-like kinases or proteins in the CLAVATA-WUSCHEL pathway. In this study, we identified three additional receptor-like kinases, ZmER1 (ZmERECTA), ZmER2 and ZmERL, and their cognate ligands, ZmEPFs (EPIDERMAL PATTERNING FACTOR), which play critical roles in regulating meristem activity and are essential for plant architecture and ear development. Utilizing CRISPR/Cas9 gene-editing technology, we generated ZmERs knockout mutants. The Zmer1 mutant displayed a compact plant architecture, an enlarged inflorescence meristem (IM), and increased kernel row number, while Zmer2 and Zmerl mutants showed no obvious defects. Double mutants of Zmer1; Zmer2 and Zmer1; Zmerl and triple mutants showed more severe plant architecture defects and larger IM meristem size compared to Zmer1 single mutant, suggesting redundant roles of ZmERs genes, with ZmER1 playing a dominant role. Microscale thermophoresis experiments showed that ZmER1 binds specifically to several ZmEPF peptides but not the CLE peptides. CRISPR knock-outs of five ZmEPF genes revealed that single mutant of ZmEPF showed no obvious phenotypes, but higher-order mutants exhibited significantly enlarged IM, suggesting functional redundancy among the ZmEPFs. Notably, unlike the Zmers mutant with both plant architecture and ear phenotype, quintyple Zmepfs displayed no plant architecture defect, implying a specific role for ZmEPFs in inflorescence meristem regulation and a function uncoupling between the receptor and its ligand. Interestingly, ZmER1 directly interacts with ZmCRN, a key signaling component of the CLV pathway, and &nbsp;Zmwus1 mutant suppressed the enlarged inflorescence meristem caused by Zmer1 mutation. These findings suggest that ZmERs regulate meristem development by integrating with the CLV-WUS pathway. This study provides new insights into ZmER-mediated regulation of plant and ear architecture and identifies candidate genetic targets for breeding high-yielding maize varieties through optimizing the plant and ear architecture.</p>
</div>
</div>
</div>
</section>
<section id="t16-catalytic-and-non-catalytic-trehalose-6-phosphate-synthases-tpss-interact-with-ramosa3-to-control-maize-development" class="level2">
<h2 class="anchored" data-anchor-id="t16-catalytic-and-non-catalytic-trehalose-6-phosphate-synthases-tpss-interact-with-ramosa3-to-control-maize-development">T16: Catalytic and non-catalytic TREHALOSE-6-PHOSPHATE SYNTHASES (TPSs) interact with RAMOSA3 to control maize development</h2>
<p><font color="#339966">Cell and Developmental Biology </font> <a href="mailto:tran@cshl.edu">Thu Tran</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-31-contents" aria-controls="callout-31" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-31" class="callout-31-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:Tran@cshl.edu">Tran, Thu</a><sup>1</sup><br>Claeys, Hannes<sup>2</sup><br>Michalski, Kevin<sup>1</sup><br>Boumpas, Panagiotis<sup>3</sup><br>Williams, Z'Dhanne<sup>4</sup><br>Sheppard, Samatha<sup>1</sup><br>Vi, Son<sup>5</sup><br>Chou, Tsung Han<sup>1</sup><br>Furukawa, Hiro<sup>1</sup><br>Jackson, David<sup>1</sup><br></p>
<p><sup>1</sup>Cold Spring Harbor Laboratory, 1 Bungtown Road, Cold Spring Harbor, NY11724, USA<br><sup>2</sup>Inari, Industriepark 7A, 9052 Zwijnaarde, Belgium<br><sup>3</sup>Heidelberg University, Centre for Organismal Studies (COS) Heidelberg, Department of Developmental Biology, Heidelberg, Germany<br><sup>4</sup>Salivary Disorders Unit, National Institute of Dental and Craniofacial Research, National Institute of Health, Bethesda, MD, USA<br><sup>5</sup>Faculty of Biotechnology, Chemistry and Environmental Engineering, PHENIKAA University, Yen Nghia, Ha Dong, Hanoi 12116, Vietnam<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-32-contents" aria-controls="callout-32" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-32" class="callout-32-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Trehalose-6-phosphate (T6P)&nbsp;is a key regulator of plant signaling networks, and coordinates growth by influencing carbon allocation, stress responses, architecture, and developmental transitions.&nbsp;T6P&nbsp;is the intermediate of trehalose biosynthesis mediated by T6P-synthases (TPSs) and T6P-phosphatases (TPPs).&nbsp;Plants harbor small families of&nbsp;TPS&nbsp;and&nbsp;TPP&nbsp;genes; while all TPPs are &nbsp;catalytic active, most plant TPSs are non-catalytic, suggesting they have regulatory functions.&nbsp;Here, we show that non-catalytic TPSs form a tripartite complex with catalytic TPSs and TPPs to control enzymatic activity and development.Maize mutant&nbsp;ramosa3&nbsp;(ra3) increases inflorescence branching and RA3 encodes a catalytic TPP. To investigate RA3 molecular mechanism, we screened for interactors, and found that it interacts with two non-catalytic TPS1, TPS12.&nbsp;tps1&nbsp;and&nbsp;tps12&nbsp;mutants enhance&nbsp;ra3&nbsp;phenotypes, suggesting their interaction is biologically significant. Interestingly, we found that TPS1 also interacts with the two maize catalytic TPS11 and TPS14. We knocked out these genes using CRISPR-Cas9, and the double mutants are embryo lethal. However, reducing active TPS expression in the&nbsp;ra3; tps1; tps12&nbsp;triple mutant background modifies its phenotype, supporting the idea interaction between three classes of proteins.To ask if TPS-TPP interactions affect enzyme activity, we performed a coupled enzyme assay, and found that the non-catalytic TPS1 stimulated the activity of RA3 and TPS14. This result suggests that RA3, TPS1, and TPS14 form a complex, and we confirmed after purifying the three proteins from insect cells. We used AlphaFold to predict that the TPS domains of TPS1 and TPS14 initiate this complex. To confirm this prediction, we co-expressed TPS1 and TPS14 and visualized&nbsp;heterotetramer&nbsp;complex formation by cryo-electron microscopy.&nbsp;From these results, we propose that the TPS1 TPS14 heterotetramer is important for both enzymatic activity and complex formation. These results provide&nbsp;insights into the structural basis and stoichiometry of TPS-TPP interactions, which have not been studied in any organism.In summary, we show a maize TPP (RA3) functions in a complex with both non-catalytic and catalytic active TPSs, and the non-catalytic TPS stimulates the activity of the active enzymes. Our research provides insights for the first time into the combined activity of the two major trehalose gene classes in plant development.</p>
</div>
</div>
</div>
</section>
<section id="t17-exploring-the-effects-of-ethylene-related-transcription-factors-during-maize-shoot-development" class="level2">
<h2 class="anchored" data-anchor-id="t17-exploring-the-effects-of-ethylene-related-transcription-factors-during-maize-shoot-development">T17: Exploring the effects of ethylene-related transcription factors during maize shoot development</h2>
<p><font color="#339966">Cell and Developmental Biology </font> <a href="mailto:jaaragon@iastate.edu">Alejandro Aragon Raygoza</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-33-contents" aria-controls="callout-33" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-33" class="callout-33-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:jaaragon@iastate.edu">Aragon-Raygoza, Alejandro</a><sup>1 2</sup><br>Galli, Mary<sup>3</sup><br>Best, Norman<sup>4</sup><br>Unger-Wallace, Erica<sup>2</sup><br>Satterlee, James<sup>5</sup><br>Beeler, Charlie<sup>2</sup><br>Yen, Jeffrey<sup>5</sup><br>Rasey, Oliver<sup>1</sup><br>Darst, Faith<sup>1</sup><br>Vollbrecht, Erik<sup>2</sup><br>Gallavotti, Andrea<sup>3</sup><br><a href="mailto:strable@iastate.edu">Strable, Josh</a><sup>1 2</sup><br></p>
<p><sup>1</sup>Department of Molecular and Structural Biochemistry, North Carolina State University<br><sup>2</sup>Department of Genetics, Development and Cell Biology, Iowa State University<br><sup>3</sup>The Waksman Institute of Microbiology, Department of Plant Biology, Rutgers University<br><sup>4</sup>Plant Genetics Research Unit, USDA-ARS<br><sup>5</sup>Plant Biology Section, Cornell University<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-34-contents" aria-controls="callout-34" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-34" class="callout-34-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>The plant hormone ethylene is a key developmental regulator that acts largely to restrict growth. Ethylene signaling and response are well-studied in eudicots; in contrast, our understanding of ethylene function in cereal crops is markedly limited. Here, we characterized mutations in maize ETHYLENE INSENSITIVE3(EIN3)-LIKE1 (ZmEIL1) and ZmEIL9 genes that encode co-orthologs of the Arabidopsis EIN3/EIL1 hub transcription factors. By stacking independently derived mutations in ZmEIL1 and ZmEIL9, we uncovered vegetative shoot phenotypes associated with mis-regulated growth that are distinct from Arabidopsis ein3;eil1 mutants. We show that Zmeil1;Zmeil9 double mutants are insensitive to the ethylene precursor ACC in separate germination and growth assays. By utilizing single-cell transcriptomics, we identified tissue- and cell-specific genetic signatures in Zmeil1;Zmeil9 and normal shoots with or without exogenous ACC. We found that differentially expressed genes in hormone pathways are highly correlated with altered accumulation of hormone pathway intermediates in Zmeil1;Zmeil9 shoot apices. Gene ontology analysis revealed enrichment of abiotic stress response categories in Zmeil1;Zmeil9 upregulated genes, whereas genes encoding translational machinery were downregulated in mutant shoot apices. Translation was enriched in genes upregulated by response to ACC, while developmental process and response to stimulus were significant categories in ACC-downregulated genes. We leveraged DAP-seq to identify genome-wide binding positions of ZmEILs. Approximately 85% of ZmEIL1 and ZmEIL9 bound filtered-genic regions overlap, suggesting co-regulation of a large portion of putative target genes, including a proportion of genes that were identified as differentially expressed in our single-cell data. Genes directly bound and modulated by ZmEILs include ZAR and RTL family members that are negative regulators of ethylene signaling, and that also have important roles in organ growth. Our study shows that ZmEIL genes are central regulators of shoot growth and, by resolving cellular and tissue heterogeneity of ethylene response, significantly advances our understanding of ethylene signaling in plants.</p>
</div>
</div>
</div>
</section>
<section id="t18-understanding-the-molecular-mechanism-of-parthenogenesis-in-cereals" class="level2">
<h2 class="anchored" data-anchor-id="t18-understanding-the-molecular-mechanism-of-parthenogenesis-in-cereals">T18: Understanding the molecular mechanism of parthenogenesis in cereals</h2>
<p><font color="#339966">Cell and Developmental Biology </font> <a href="mailto:Xixi.zheng@ur.de">Xixi Zheng</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-35-contents" aria-controls="callout-35" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-35" class="callout-35-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:Xixi.zheng@ur.de">Zheng, Xixi</a><sup>1</sup><br><a href="mailto:maria.flores@itqb.unl.pt">Tornero, Maria Flores</a><sup>1</sup><br><a href="mailto:Uwe.Schwartz@vkl.uni-regensburg.de">Schwartz, Uwe</a><sup>2</sup><br><a href="mailto:Thomas.Dresselhaus@biologie.uni-regensburg.de">Dresselhaus, Thomas</a><sup>1</sup><br></p>
<p><sup>1</sup>Cell Biology and Plant Biochemistry, University of Regensburg, 93053 Regensburg, Germany<br><sup>2</sup>Computational Core Unit, University of Regensburg, 93053 Regensburg, Germany<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-36-contents" aria-controls="callout-36" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-36" class="callout-36-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Parthenogenesis, describes spontaneous embryogenesis from an unfertilized egg cell and thereby generates offspring genetically identical to the mother plant, and is a key component of apomixis (asexual reproduction through seeds). Investigating parthenogenesis in crop plants not only has high potentials to immediately fix desired traits including heterosis and thus would create great economic values, but would also help to understand how egg cell fate is determined for embryogenesis initiation. The underlying mechanisms of parthenogenesis remain poorly understood. Here, we use the apomictic grass&nbsp;Tripsacum dactyloides&nbsp;to address these questions. As the closest wild relative of maize,&nbsp;Tripsacum&nbsp;is sexually reproducing as a diploid, but all polyploids display apomixis via parthenogenesis. Wecollected egg cells from diploid and tetraploid Tripsacum&nbsp;lines to compare their gene expression. We observed that parthenogenetic eggs possess relatively specific cell cycle gene expression pattern that confers division potentials. Transcriptional reprogramming considerably contributes via both ON/OFF and differential regulation modes primarily involved in cell differentiation and auxin signaling. Parthenogenesis and zygotic genome activation share similar gene expression alterations associated with RNA metabolisms at both transcriptional (e.g.&nbsp;via ZmBBM1) and post-transcriptional (e.g.&nbsp;via RNase exonuclease) levels. These genes are&nbsp;highly expressed in parthenogenetic eggs&nbsp;but completely&nbsp;silenced&nbsp;in sexual eggs. We&nbsp;are currently characterizing&nbsp;the funtion(s) of candidate genes&nbsp;via creation ofectopic sexual egg cell-expression lines&nbsp;and&nbsp;knock-outs&nbsp;in&nbsp;maize. We found both ectopic expression&nbsp;of&nbsp;ZmBBM1&nbsp;or knock out can induced&nbsp;twin embryos.&nbsp;ZmBBM1-mEGFP fluorescence are abundantly detected in embryonic pro-vascular and root systems. The ultimate goal of this research is to gain a mechanistic understanding of parthenogenesis and embryogenesis initiation in cereals and to utilize the knowledge generated to contribute and improve the production of haploid maize lines and/or clonal seeds.</p>
</div>
</div>
</div>
</section>
<section id="t19-heat-treatment-and-uba2-fusions-enhance-lbcas12a-genome-editing-activity-during-haploid-induction" class="level2">
<h2 class="anchored" data-anchor-id="t19-heat-treatment-and-uba2-fusions-enhance-lbcas12a-genome-editing-activity-during-haploid-induction">T19: Heat treatment and UBA2 fusions enhance LbCas12a genome editing activity during haploid induction</h2>
<p><font color="#339966">Cell and Developmental Biology </font> <a href="mailto:rachel.egger@syngenta.com">Rachel Egger</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-37-contents" aria-controls="callout-37" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-37" class="callout-37-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:dawei.liang@syngenta.com">Liang, Dawei</a><sup>1</sup><br><a href="mailto:fugui.zhu@syngenta.com">Zhu, Fugui</a><sup>1</sup><br><a href="mailto:juan.wei@syngenta.com">Wei, Juan</a><sup>1</sup><br><a href="mailto:huan.guo@syngenta.com">Guo, Huan</a><sup>1</sup><br><a href="mailto:yuguo.zhang@syngenta.com">Zhang, Yuguo</a><sup>1</sup><br><a href="mailto:julie.green@syngenta.com">Green, Julie</a><sup>2</sup><br><a href="mailto:yun.ji@syngenta.com">Ji, Yun</a><sup>1</sup><br><a href="mailto:huaibing.jin@syngenta.com">Jin, Huaibing</a><sup>1</sup><br><a href="mailto:xiujuan.zhang@syngenta.com">Zhang, Xiujuan</a><sup>1</sup><br><a href="mailto:yubo.liu@syngenta.com">Liu, Yubo</a><sup>1</sup><br><a href="mailto:yu.zhang@syngenta.com">Zhang, Yu</a><sup>1</sup><br><a href="mailto:han.wang@syngenta.com">Wang, Han</a><sup>1</sup><br><a href="mailto:xi.chen@syngenta.com">Chen, Xi</a><sup>1</sup><br><a href="mailto:tim.kelliher@syngenta.com">Kelliher, Tim</a><sup>2</sup><br><a href="mailto:rachel.egger@syngenta.com">Egger, Rachel</a><sup>2</sup><br></p>
<p><sup>1</sup>State Key Laboratory of Crop Germplasm Innovation and Molecular Breeding, Syngenta Biotechnology (China) Co., Ltd, Beijing, China<br><sup>2</sup>Syngenta Seeds Research, Research Triangle Park, NC, USA<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-38-contents" aria-controls="callout-38" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-38" class="callout-38-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Haploid induction (HI) coupled with genome editing, known as HI-Edit, has emerged as a powerful tool for direct genomic modification of commercial crop varieties including maize, eliminating the need for direct variety transformation of CRISPR machinery. However, the efficiency of this method has been limited by relatively low haploid editing rates (HER). In this study, we report significant improvements to HI-Edit efficiency through the application of post-pollination heat treatment, LbCas12a expression with a male-gamete-specific promoter, and the use of a LbCas12a-UBA2 fusion protein to promote protein stability during HI. Our results demonstrate that heat treatment increased haploid editing rates by 10-fold, providing a simple yet effective means to enhance the yield of edited haploid plants. Furthermore, we found that UBA2 fusion can further increase haploid editing rate by 4-fold, offering an additional strategy to boost editing efficiency. The combination of heat treatment and UBA2 fusion resulted in a cumulative 16-fold increase in haploid editing rates, significantly outperforming traditional HI-Edit protocols. In this experiment, an average of 25% of all haploids were successfully edited across two testers at a single gRNA target site. Our findings provide a robust framework for expanding the scalability of this technology in maize and potentially across diverse plant species, enhancing the efficiency of HI-Edit and accelerating crop improvement.</p>
</div>
</div>
</div>
</section>
<section id="t20-molecular-evolution-of-the-ga-reproductive-barriers-in-maize-and-related-species" class="level2">
<h2 class="anchored" data-anchor-id="t20-molecular-evolution-of-the-ga-reproductive-barriers-in-maize-and-related-species">T20: Molecular evolution of the Ga reproductive barriers in maize and related species</h2>
<p><font color="#FF0066">Evolution and Population Genetics</font> <a href="mailto:epcryan@ucdavis.edu">Elli Cryan</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-39-contents" aria-controls="callout-39" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-39" class="callout-39-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:epcryan@ucdavis.edu">Cryan, Elli</a><sup>1 2 3</sup><br>Phinney, Garnet<sup>2</sup><br>Seetharam, Arun S.<sup>4 5</sup><br>Evans, Matthew M.S.<sup>6</sup><br>Kellogg, Elizabeth A.<sup>7 8</sup><br>Zhan, Junpeng<sup>7</sup><br>Meyers, Blake C.<sup>1 7 9 10</sup><br><a href="mailto:kliebenstein@ucdavis.edu">Kliebenstein, Daniel J.</a><sup>1 3 10</sup><br><a href="mailto:rossibarra@ucdavis.edu">Ross-Ibarra, Jeffrey</a><sup>2 3 10</sup><br></p>
<p><sup>1</sup>Plant Sciences, UC Davis; Davis, CA<br><sup>2</sup>Evolution and Ecology, UC Davis; Davis, CA<br><sup>3</sup>Center for Population Biology, UC Davis; Davis, CA<br><sup>4</sup>Ecology, Evolution and Organismal Biology, Iowa State University; Ames, IA<br><sup>5</sup>Genetics, Development and Cell Biology, Iowa State University; Ames, IA<br><sup>6</sup>Plant Biology, Carnegie Institution for Science; Stanford, CA<br><sup>7</sup>Donald Danforth Plant Science Center; St.&nbsp;Louis, MO<br><sup>8</sup>Arnold Arboretum of Harvard University; Boston, MA<br><sup>9</sup>Plant Science &amp; Technology, University of Missouri - Columbia; Columbia, MO<br><sup>10</sup>Genome Center, UC Davis; Davis, CA<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-40-contents" aria-controls="callout-40" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-40" class="callout-40-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Three cross-incompatibility loci, Teosinte crossing barrier1 (Tcb1), Gametophytic factor1 (Ga1), and Ga2, each play a key role in preventing hybridization between incompatible populations and are proposed to maintain the barrier between domesticated maize and wild teosinte subspecies. Each locus encodes both a silk-active and a matching pollen-active pectin methylesterase (PME). To investigate the diversity and molecular evolution of these gametophytic factor loci, we identified existing and improved models of the responsible genes in a new genome assembly of maize line P8860 that contains active versions of all three loci. We then examined fifty-two assembled genomes from seventeen species to classify haplotype diversity and identify sites under diversifying selection during the evolution of these genes. We show that Ga2, the oldest of these three loci, was duplicated to form Ga1 at least 12 million years ago. Tcb1, the youngest locus, arose as a duplicate of Ga1 before or around the time of diversification of the Zea genus. We find evidence of positive selection during evolution of the functional genes at an active site in the pollen-expressed PME and predicted surface sites in both the silk- and pollen-expressed PMEs. The most common allele at the Ga1 locus is a conserved ga1 allele (ga1-Off), which is specific haplotype containing three full-length PME gene copies, all of which are non-coding due to conserved stop codons and are between 610 thousand and 1.5 million years old. We show that the ga1-Off allele likely generates 24-nt siRNAs in developing pollen-producing tissue, and these siRNAs map to functional Ga1 alleles. In crossing experiments, the ga1-Off allele was associated with reduced strength of the typically dominant functional alleles for the Ga1 and Tcb1 barriers. Taken together, this seems to be an example of paramutation functioning at a locus controlling a reproductive barrier.</p>
</div>
</div>
</div>
</section>
<section id="t21-long-distance-retrotransposons-direct-variable-gene-imprinting-in-maize" class="level2">
<h2 class="anchored" data-anchor-id="t21-long-distance-retrotransposons-direct-variable-gene-imprinting-in-maize">T21: Long-distance retrotransposons direct variable gene imprinting in maize</h2>
<p><font color="#0066CC">Transposons &amp; Epigenetics </font> <a href="mailto:qi.li@zmbp.uni-tuebingen.de">Qi Li</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-41-contents" aria-controls="callout-41" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-41" class="callout-41-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Li, Qi<sup>1 2</sup><br>Zheng, Xixi<sup>1 3</sup><br>Li, Changsheng<sup>1 4</sup><br>Wu, Yongrui<sup>1</sup><br></p>
<p><sup>1</sup>National Key Laboratory of Plant Molecular Genetics, CAS Center for Excellence in Molecular Plant Sciences, Shanghai Institute of Plant Physiology &amp; Ecology, Chinese Academy of Sciences, Shanghai 200032, China<br><sup>2</sup>Center for Plant Molecular Biology, University of T√ºbingen, T√ºbingen 72076, Germany<br><sup>3</sup>Cell Biology and Plant Biochemistry, Institute of Plant Sciences, University of Regensburg, Regensburg 93053, Germany<br><sup>4</sup>The National Engineering Laboratory of Crop Stress Resistance Breeding, Anhui Agricultural University, Hefei 230036, China.<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-42-contents" aria-controls="callout-42" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-42" class="callout-42-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Genomic imprinting dictates the preferential expression of parental alleles based on their origin, rather than dosage. Despite decades of research, understanding how imprint control arises or is removed at specific loci through regulatory regions, and how to shape the widespread intra-species imprint variability, remains limited. Here, we developed a genetic screening system utilizing a classic maize endosperm mutant that exhibits maternally inherited dominant-negative effects due to conserved paternal imprints on the causal locus. By screening an inbred line panel and sequencing the genome of a specific inbred, we identified rare haplotypes that bypass imprinting. We discovered that the conserved locus-specific imprinting is maintained by a&nbsp;Flip&nbsp;retrotransposon located hundred kilobases upstream, which has persisted throughout maize domestication. The&nbsp;Xilon-Diguus&nbsp;and&nbsp;Cinful-Zeon&nbsp;impede imprinting at the locus by inserting into the&nbsp;Flip, forming nested retrotransposon structures.&nbsp;Our findings shed light on how single, distant retrotransposons can both preserve and perturb gene imprinting through proximal-distal interactions. Our research highlights the pivotal role of hidden, often overlooked distal sequences‚Äîparticularly transposable element nesting‚Äîin driving fluctuations in gene expression within natural populations.</p>
</div>
</div>
</div>
</section>
<section id="t22-deciphering-epigenetic-and-genetic-alterations-in-a-dna-methylation-mutant-through-successive-generations-of-self-fertilization-in-maize" class="level2">
<h2 class="anchored" data-anchor-id="t22-deciphering-epigenetic-and-genetic-alterations-in-a-dna-methylation-mutant-through-successive-generations-of-self-fertilization-in-maize">T22: Deciphering epigenetic and genetic alterations in a DNA methylation mutant through successive generations of self-fertilization in maize</h2>
<p><font color="#0066CC">Transposons &amp; Epigenetics </font> <a href="mailto:xicheng@ufl.edu">Xi Cheng</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-43-contents" aria-controls="callout-43" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-43" class="callout-43-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:xicheng@ufl.edu">Cheng, Xi</a><sup>1 2</sup><br><a href="mailto:meixiazhao@ufl.edu">Zhao, Meixia</a><sup>1 2</sup><br><a href="mailto:gwonjin.lee@wvstateu.edu">Lee, Gwonjin</a><sup>3</sup><br></p>
<p><sup>1</sup>Plant Molecular and Cellular Biology Program, University of Florida, Gainesville, FL 32610, USA<br><sup>2</sup>Department of Microbiology and Cell Science, University of Florida, Gainesville, FL 32611, USA<br><sup>3</sup>Department of Biology, West Virginia State University, Institute, WV 25112, USA<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-44-contents" aria-controls="callout-44" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-44" class="callout-44-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Enhancing crop productivity in the face of global challenges relies on a deeper understanding of the genetic and epigenetic mechanisms governing plant growth and development. In particular, how plants maintain genome stability and regulate gene expression under epigenetic changes is becoming increasingly critical.&nbsp;Our research focuses on maize plants mutant for a small RNA biosynthesis pathway gene, named&nbsp;Mop1&nbsp;(Mediator of paramutation1). These&nbsp;mop1&nbsp;mutants exhibit increasingly severe phenotypes related to growth and development after several generations of self-fertilization in inbred plants,&nbsp;a phenomenon not observed in wild-type plants. Our DNA methylation analysis of&nbsp;mop1&nbsp;mutants homozygous for three generations revealed that while genome-wide CG and CHG (where H = A, T, or C) methylation levels remained relatively stable, CHH methylation was rapidly removed in the first-generation mutants, with no further reductions in subsequent generations. Interestingly, despite the global stability of CG and CHG methylation, the numbers of both hyper- and hypomethylated CG and CHG differentially methylated regions (DMRs) increased dramatically over successive generations, indicating substantial local changes. While hypermethylated DMRs accounted for only 10% of the total, a significant proportion of these, particularly CHG hyper-DMRs, were enriched within introns of genes overlapping long interspersed nuclear elements (LINEs). To validate our findings and test hypotheses concerning the role of polymorphic transposable elements (TEs) in these epigenetic changes, we analyzed&nbsp;mop1&nbsp;mutants homozygous for multiple generations in the Mo17 background. The results in Mo17 mirrored those in B73, with increasing DMRs in non-CHH contexts and similar epigenetic changes, indicating that these patterns are not specific to a particular maize genotype. Our next steps involve exploring gene expression, chromatin accessibility, and histone modifications in these plants, focusing on genes or regions at or near those DMRs. By uncovering the interplay between DNA methylation, differential expression of genes or TEs, and chromatin structures, we aim to elucidate the genetic and epigenetic mechanisms driving maize development. These insights hold practical significance, enabling innovative epigenetic engineering approaches to enhance maize productivity and performance while advancing the understanding of plant epigenetic regulation.</p>
</div>
</div>
</div>
</section>
<section id="t23-replication-timing-uncovers-a-novel-two-compartment-arrangement-of-maize-interphase-euchromatin" class="level2">
<h2 class="anchored" data-anchor-id="t23-replication-timing-uncovers-a-novel-two-compartment-arrangement-of-maize-interphase-euchromatin">T23: Replication timing uncovers a novel two-compartment arrangement of maize interphase euchromatin</h2>
<p><font color="#669933">Cytogenetics</font> <a href="mailto:ha20be@fsu.edu">Hafiza Sara Akram</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-45-contents" aria-controls="callout-45" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-45" class="callout-45-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:ha20be@fsu.edu">Akram, Hafiza Sara</a><sup>1</sup><br><a href="mailto:emily_wear@ncsu.edu">Wear, Emily E.</a><sup>2</sup><br><a href="mailto:lamickel@ncsu.edu">Mickelson-Young, Leigh</a><sup>2</sup><br><a href="mailto:zachturpin92@gmail.com">Turpin, Zach M.</a><sup>1</sup><br><a href="mailto:wftb@ncsu.edu">Thompson, William F.</a><sup>2</sup><br><a href="mailto:lkhanley@ncsu.edu">Hanley-Bowdoin, Linda K.</a><sup>2</sup><br><a href="mailto:lconcia@tacc.utexas.edu">Concia, Lorenzo</a><sup>3</sup><br><a href="mailto:bass@bio.fsu.edu">Bass, Hank W.</a><sup>1</sup><br></p>
<p><sup>1</sup>Department of Biological Science, Florida State University, Tallahassee, FL, USA<br><sup>2</sup>Plant and Microbial Biology, North Carolina State University, Raleigh, NC, USA<br><sup>3</sup>Texas Advanced Computing Center, University of Texas, Austin, TX, USA<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-46-contents" aria-controls="callout-46" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-46" class="callout-46-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>The time within S phase when a given genomic region replicates is measurable and referred to as Replication Timing (RT). We previously proposed a ‚Äúmini-domain‚Äù chromatin fiber RT model for maize euchromatin‚Äôs spatial architecture. 3D quantitative cytology showed that euchromatin is subdivided into two compartments during S-phase, distinguished by chromatin condensation and replication timing: Early-S and Middle-S. A key gap remains in understanding whether this compartmentalization is a general feature throughout the cell cycle, which could greatly impact our knowledge of genome architecture and gene regulation. To investigate this, we conducted two orthogonal assays, Hi-C for genome-wide interaction data and 3D FISH for direct visualization of chromatin organization. The Hi-C-derived eigenvalues and insulation scores significantly concord with Early-S and Middle-S regions. Early-S regions showed negative insulation scores with more long-range contacts, whereas Middle-S regions displayed the opposite, positive insulation scores with fewer long-range contacts. Compared to Middle-S regions, the Early-S regions showed much stronger correlations with epigenomic signatures of open and transcriptionally active chromatin. Oligo FISH painting with RT-specific probes demonstrated that Early-S and Middle-S regions occupied adjacent but largely non-overlapping nucleoplasmic regions throughout all stages of interphase, including G1. These findings validate our model while establishing that the global ‚ÄúA‚Äù compartment of euchromatin actually consists of two spatially and epigenetically distinct substructures ‚Äì now referred to as Early-S and Middle-S compartments. Given the strong correlation between replication timing and Hi-C data from root tips, we examined the conservation of Hi-C architecture in both root tip and earshoot tissues and found them to be remarkably similar. Overall, our findings highlight the importance of replication timing as a conserved feature of chromatin architecture, reflecting broader principles of genome organization.&nbsp;</p>
</div>
</div>
</div>
</section>
<section id="t24-determination-of-genetic-and-epigenetic-regulations-of-meiotic-recombination-during-domestication-in-maize" class="level2">
<h2 class="anchored" data-anchor-id="t24-determination-of-genetic-and-epigenetic-regulations-of-meiotic-recombination-during-domestication-in-maize">T24: Determination of genetic and epigenetic regulations of meiotic recombination during domestication in maize</h2>
<p><font color="#FF0066">Evolution and Population Genetics</font> <a href="mailto:akwasiyeboah@ufl.edu">Akwasi Yeboah</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-47-contents" aria-controls="callout-47" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-47" class="callout-47-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:akwasiyeboah@ufl.edu">Yeboah, Akwasi</a><sup>1</sup><br><a href="mailto:gwonjin.lee@wvstateu.edu">Lee, Gwonjin</a><sup>2</sup><br><a href="mailto:sherry.flint-garcia@usda.gov">Flint-Garcia, Sherry A.</a><sup>3</sup><br><a href="mailto:meixiazhao@ufl.edu">Zhao, Meixia</a><sup>1</sup><br></p>
<p><sup>1</sup>University of Florida, Gainesville, FL 32603<br><sup>2</sup>West Virginia State University, Institute, WV 25112<br><sup>3</sup>University of Missouri, Columbia, MO 65211<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-48-contents" aria-controls="callout-48" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-48" class="callout-48-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Meiotic recombination involves the exchange of genetic material between homologous chromosomes, playing a key role in evolution and genetic diversity. Meiotic crossovers (COs) are not evenly distributed on chromosomes; instead, they are enriched in hotspots controlled by genetic and epigenetic factors. In maize, meiotic recombination has significantly contributed to maize domestication. However, the specific mechanisms underlying this process remain largely unexplored. This study aims to elucidate the genetic mechanisms driving meiotic recombination during maize domestication and to compare how meiotic recombination differs between sexes. Our investigation encompasses a diverse set of maize lines, including 5 teosintes, 7 landraces, and 15 cultivars. These lines were selected because they maximize genetic diversity and thus are most likely to show variation in recombination. Our results reveal that the teosinte line BK4 and the landrace MR19 exhibited longer genetic distances and higher crossover numbers, whereas BK1 and MR11 showed the lowest, compared to the other teosinte and landrace lines. Of the 27 total lines including cultivars, BK4 and MR19 were still among the highest in crossover numbers, while BK1 and CML103 showed the lowest. CO hotspot analysis revealed that most hotspots were located at the distal ends of each chromosome across all lines. Additionally, our results reveal significant differences in CO number and recombination rate between sexes, and these differences were dependent on different maize genetic backgrounds. Furthermore, analysis of distance CoC showed that teosinte lines exhibited higher, though not statistically significant, levels of CO interference compared to other maize lines. In our ongoing research, we are collecting immature anthers from maize plants exhibiting extreme CO numbers and recombination rates for transcriptome and DNA methylation analysis. This work aims to uncover the genetic and epigenetic factors underlying meiotic recombination in maize, thereby enhancing our understanding of this crucial biological process.</p>
</div>
</div>
</div>
</section>
<section id="t25-deciphering-genetic-architecture-of-stalk-lodging-resistance-using-high-density-phenotype-map-in-maize" class="level2">
<h2 class="anchored" data-anchor-id="t25-deciphering-genetic-architecture-of-stalk-lodging-resistance-using-high-density-phenotype-map-in-maize">T25: Deciphering genetic architecture of stalk lodging resistance using high-density phenotype map in maize</h2>
<p><font color="#336699">Biochemical and Molecular Genetics </font> <a href="mailto:bkundur@clemson.edu">Bharath Kunduru</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-49-contents" aria-controls="callout-49" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-49" class="callout-49-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:bkundur@clemson.edu">Kunduru, Bharath</a><sup>1</sup><br><a href="mailto:norbert@qgg.au.dk">Bokros, Norbert</a><sup>2 3</sup><br><a href="mailto:taba4048@vandals.uidaho.edu">Tabaracci, Kaitlin</a><sup>4</sup><br><a href="mailto:mohank@clemson.edu">Kumar, Rohit</a><sup>1 5</sup><br><a href="mailto:mbrar@clemson.edu">Brar, Manwinder</a><sup>1</sup><br><a href="mailto:caique.m.silva@ufv.br">Machado e Silva, Caique</a><sup>6 7</sup><br><a href="mailto:odun3492@vandals.uidaho.edu">Oduntan, Yusuf</a><sup>4</sup><br><a href="mailto:deko6395@vandals.uidaho.edu">DeKold, Joseph</a><sup>4</sup><br><a href="mailto:joseph.woomer@uky.edu">Woomer, Joseph</a><sup>3</sup><br><a href="mailto:ehbisho@g.clemson.edu">Bishop, Rebecca</a><sup>1</sup><br><a href="mailto:virginia.verges@uky.edu">Verges, Virginia</a><sup>3</sup><br><a href="mailto:cstubbs@fdu.edu">Stubbs, Christopher</a><sup>4 8</sup><br><a href="mailto:morota@vt.edu">Morota, Gota</a><sup>6</sup><br><a href="mailto:mcmaha2@clemson.edu">McMahan, Christopher</a><sup>9</sup><br><a href="mailto:danieljr@uidaho.edu">Robertson, Daniel</a><sup>4</sup><br><a href="mailto:sdebo2@uky.edu">DeBolt, Seth</a><sup>3</sup><br><a href="mailto:sekhon@clemson.edu">Sekhon, Rajandeep</a><sup>1</sup><br></p>
<p><sup>1</sup>Department of Genetics and Biochemistry, Clemson University, SC, USA<br><sup>2</sup>Center for Quantitative Genetics and Genomics, Aarhus University, Aarhus, Denmark<br><sup>3</sup>Department of Horticulture, University of Kentucky, KY, USA<br><sup>4</sup>Department of Mechanical Engineering, University of Idaho, ID, USA<br><sup>5</sup>Department of Plant and Environmental Sciences, Clemson University, SC, USA<br><sup>6</sup>School of Animal Sciences, Virginia Polytechnic Institute and State University, VA, USA<br><sup>7</sup>Department of Agronomy, Federal University of Vi√ßosa, Minas Gerais, Brazil<br><sup>8</sup>School of Computer Sciences and Engineering, Fairleigh Dickinson University, NJ, USA<br><sup>9</sup>School of Mathematics and Statistical sciences, Clemson University, SC, USA<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-50-contents" aria-controls="callout-50" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-50" class="callout-50-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Stalk lodging undermines crop productivity and causes global annual losses of at least 6 billion USD in maize (Zea mays L.). Poor understanding of plant phenotypes associated with stalk lodging, referred to as intermediate phenotypes, and lack of standardized phenotyping protocols impeded the efforts to enhance genetic resolution of stalk lodging resistance. We generated a multi-environment high-density phenotype map of 15 morphological, geometric, structural, and biomechanical phenotypes on 31,200 stalks of a maize inbred panel, examined the impact of these phenotypes on stalk lodging resistance, and identified underlying genetic loci. Preserving field location of individual plants allowed us to account for spatial effects and refine the phenotype data for statistical analyses. Phenotype analyses revealed significant variation across environments indicating a substantial role of G√óE on the phenotypic continuum of intermediate phenotypes. Predictive analytics with machine learning revealed major and minor diameters of internodes and plant height as the key predictors of stalk flexural stiffness. Intermediate phenotypes measured on the lower-most elongated internode showed stronger genetic correlation with stalk flexural stiffness as compared to those measured on the primary ear-bearing internode. Most intermediate phenotypes exhibited low to moderate heritability indicating low genetic tractability and complex genetic inheritance. Genome wide association analyses of the intermediate phenotypes showed candidate genes involved in cell division, plant and ear height, electron transport, membrane structure and transport, oxidoreductase activity, transcription factors, etc. Remarkably, a large proportion of the significant SNPs identified were nested in the non-coding regions of the genome and certain SNPs were shared between different phenotypes indicating pleiotropic regulation of stalk lodging resistance. We are currently evaluating the role of selected candidate genes for their role in stalk lodging resistance in maize.</p>
</div>
</div>
</div>
</section>
<section id="t26-aaf-plasticity-and-fitness-trade-offs-in-switchgrass-revealed-by-open-science-and-citizen-science-data" class="level2">
<h2 class="anchored" data-anchor-id="t26-aaf-plasticity-and-fitness-trade-offs-in-switchgrass-revealed-by-open-science-and-citizen-science-data">T26: AAF Plasticity and fitness trade-offs in switchgrass revealed by open science and citizen science data</h2>
<p><font color="#663399">Computational and Large-Scale Biology </font> <a href="mailto:laura.tibbs-cortes@usda.gov">Laura Tibbs-Cortes</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-51-contents" aria-controls="callout-51" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-51" class="callout-51-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:laura.tibbs-cortes@usda.gov">Tibbs-Cortes, Laura E</a><sup>1 2</sup><br><a href="mailto:jbjewell@wsu.edu">Jewell, Jeremy</a><sup>3</sup><br><a href="mailto:laura.bartley@wsu.edu">Bartley, Laura</a><sup>3</sup><br><a href="mailto:kswaminathan@hudsonalpha">Swaminathan, Kankshita</a><sup>4</sup><br><a href="mailto:carson.andorf@usda.go">Andorf, Carson</a><sup>1 2</sup><br><a href="mailto:tjuenger@austin.utexas.edu">Juenger, Thomas</a><sup>5</sup><br><a href="mailto:jmyu@iastate.edu">Yu, Jianming</a><sup>2</sup><br><a href="mailto:li.xianran@usda.gov">Li, Xianran</a><sup>3 6</sup><br></p>
<p><sup>1</sup>USDA-ARS Corn Insects and Crop Genetics Research Unit, Ames, IA, USA 50011<br><sup>2</sup>Iowa State University, Ames, IA, USA 50011<br><sup>3</sup>Washington State University, Pullman, WA, USA 99164<br><sup>4</sup>HudsonAlpha Institute for Biotechnology, Huntsville, AL, USA 35806<br><sup>5</sup>University of Texas at Austin, Austin, TX, USA 78712<br><sup>6</sup>USDA-ARS Wheat Health, Genetics, and Quality Research Unit, Pullman, WA, USA 99164<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-52-contents" aria-controls="callout-52" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-52" class="callout-52-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Switchgrass (Panicum virgatum), a member of the Panicoideae subfamily which includes maize, is a perennial grass native to the North American tallgrass prairie that today is grown as a biomass crop for bioenergy. Switchgrass provides a unique opportunity to study adaptation and phenotypic plasticity across a large environmental gradient because it not only has a wealth of open science genomic and phenotypic data available from multi-environment trials (METs), but is also widely observed by citizen scientists across its extensive native range. Based on thousands of research-grade observations from the citizen science repository iNaturalist, we identified a conserved trend in flowering time across latitudes. We then applied the CERIS-JGRA algorithm and QTL mapping to open science MET data in order to identify specific genetic and environmental factors influencing flowering time as well as biomass and over-winter survival rates. We found that these traits were strongly influenced by temperature and identified three major candidate genes underlying this plastic response to the environment. Candidate genes are currently being validated by CRISPR. Intriguingly, by aligning the switchgrass proteome to the maize proteome, we found that two of these three candidate genes were also recently published as candidate genes underlying plasticity in maize. Alternative haplotypes at these loci differ substantially in their degree of phenotypic plasticity in response to identified environmental cues, resulting in fitness trade-offs across the native range of switchgrass. We combined our CERIS-JGRA models with past weather data and future climate projections to model shifts in the distribution of switchgrass populations over time. Finally, by uncovering and resolving apparently opposite trends in flowering time between the citizen science and MET data, we provide insights for interpretation of these data types while leveraging their complementary strengths.</p>
</div>
</div>
</div>
</section>
<section id="t27-inactivation-of-a-lysine-histidine-transporter-1-gene-confers-southern-leaf-blight-resistance-in-maize" class="level2">
<h2 class="anchored" data-anchor-id="t27-inactivation-of-a-lysine-histidine-transporter-1-gene-confers-southern-leaf-blight-resistance-in-maize">T27: Inactivation of a lysine-histidine transporter-1 gene confers southern leaf blight resistance in maize</h2>
<p><font color="#996633">Quantitative Genetics &amp; Breeding</font> <a href="mailto:qyang@nwafu.edu.cn">Qin Yang</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-53-contents" aria-controls="callout-53" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-53" class="callout-53-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Wang, Yapeng<sup>1</sup><br>Huang, Xiaojian<sup>1</sup><br>Tian, Yuan<sup>1</sup><br>Wang, Zhe<sup>1</sup><br>Tai, Huanhuan<sup>1</sup><br>Song, Weibin<sup>2</sup><br>Balint-Kurti, Peter<sup>3 4</sup><br>Yang, Qin<sup>1</sup><br></p>
<p><sup>1</sup>State Key Laboratory for Crop Stress Resistance and High-Efficiency Production, College of Agronomy, Northwest A&amp;F University, Yangling 712100, P. R. China<br><sup>2</sup>National Maize Improvement Center, Department of Plant Genetics and Breeding, China Agricultural University, Beijing 100193, P. R. China<br><sup>3</sup>Department of Entomology and Plant Pathology, North Carolina State University, Raleigh, NC 27695, USA<br><sup>4</sup>Plant Science Research Unit, USDA-ARS, Raleigh 27695, NC, USA<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-54-contents" aria-controls="callout-54" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-54" class="callout-54-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Southern leaf blight (SLB) is one of the most serious foliar diseases in maize worldwide. qSLB6.01 is a major quantitative trait locus conferring recessive resistance to SLB. Through map-based cloning, ethyl methanesulfonate mutagenesis, and CRISPR-Cas9 editing, we demonstrate that a lysine-histidine transporter-1 (ZmLHT1) gene at qSLB6.01 confers quantitative susceptibility to SLB. A 354 bp insertion in the ZmLHT1 coding region in the resistant parental line NC292 creates a truncated protein, resulting in enhanced disease resistance to SLB. Targeted mutation of ZmLHT1 leads to robust SLB resistance without affecting other important agronomic traits. ZmLHT1 encodes a plasma membrane-localized broad-spectrum amino acid transporter. Transcriptome profiling reveals that genes involved in plant secondary cell wall biosynthesis were strongly induced in leaves of the zmlht1 mutant after Cochliobolus heterostrophus infection, whilst cell redox homeostasis-related genes were highly expressed in wildtype plants. Moreover, we present evidence that ZmLHT1 reduces ROS deposition and inhibits secondary cell wall thickening during C. heterostrophus infection. Our findings may aid in disease resistance breeding through marker-assisted selection or genome editing while balancing growth-defense tradeoffs in maize.</p>
</div>
</div>
</div>
</section>
<section id="t28-regulation-of-heterosis-associated-gene-expression-complementation-in-maize-hybrids" class="level2">
<h2 class="anchored" data-anchor-id="t28-regulation-of-heterosis-associated-gene-expression-complementation-in-maize-hybrids">T28: Regulation of heterosis-associated gene expression complementation in maize hybrids</h2>
<p><font color="#996633">Quantitative Genetics &amp; Breeding</font> <a href="mailto:mpitz@uni-bonn.de">Marion Pitz</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-55-contents" aria-controls="callout-55" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-55" class="callout-55-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:mpitz@uni-bonn.de">Pitz, Marion</a><sup>1</sup><br>Baldauf, Jutta A.<sup>1</sup><br>Piepho, Hans-Peter<sup>2</sup><br>Yu, Peng<sup>3</sup><br>Schoof, Heiko<sup>4</sup><br>Mason, Annaliese S.<sup>5</sup><br>Li, Guoliang<sup>6</sup><br><a href="mailto:hochhold@uni-bonn.de">Hochholdinger, Frank</a><sup>1</sup><br></p>
<p><sup>1</sup>Institute of Crop Science and Resource Conservation, Crop Functional Genomics, University of Bonn, 53113 Bonn, Germany<br><sup>2</sup>Institute of Crop Science, Biostatistics Unit, University of Hohenheim, 70599 Stuttgart, Germany<br><sup>3</sup>Institute of Crop Science and Resource Conservation, Emmy Noether Group Root Functional Biology, University of Bonn, 53113 Bonn, Germany<br><sup>4</sup>Institute of Crop Science and Resource Conservation, Crop Bioinformatics, University of Bonn, 53115 Bonn, Germany<br><sup>5</sup>Institute of Crop Science and Resource Conservation, Plant Breeding Department, University of Bonn, 53115, Bonn, Germany<br><sup>6</sup>Leibniz Institute of Plant Genetics and Crop Plant Research (IPK) Gatersleben, 06466 Seeland, Germany<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-56-contents" aria-controls="callout-56" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-56" class="callout-56-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Classical concepts of heterosis attribute the superiority of F1-hybrids over their homozygous parents to the complementation of unfavorable by beneficial alleles (dominance) or to heterozygote advantage (overdominance). Here we analyzed backcross hybrids of maize recombinant inbred lines, with an average heterozygosity of ~50%. This genetic architecture allowed us to study the influence of homozygous and heterozygous genomic regions on gene expression in hybrids. We demonstrated, that up to 29% of the heterotic variance in these hybrids is explained by single parent expression (SPE) complementation. In this mode of expression, consistent with the dominance model, genes are expressed in only one of the parents and in the hybrid. Furthermore, we demonstrated that eQTL regulating SPE genes are predominantly located in heterozygous regions of the genome. Thus, we demonstrated that dominance of SPE genes is important for gene activity, while heterozygosity is instrumental for the regulation of these genes. Finally, we identified an SPE gene that regulates lateral root density in hybrids. Remarkably, the activity of this gene depends on the presence of a Mo17 allele in an eQTL that regulates this gene, supporting the notion that the genetic constitution of distant regulatory elements plays a key role for the activity of heterosis-associated genes. In summary, the prevalence of dominance at the level of gene activity and overdominance at the level of gene regulation reconciles these classical genetic concepts and explains how they could both contribute to heterosis.</p>
</div>
</div>
</div>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const disableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'prefetch';
    }
  }
  const enableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'stylesheet';
    }
  }
  const manageTransitions = (selector, allowTransitions) => {
    const els = window.document.querySelectorAll(selector);
    for (let i=0; i < els.length; i++) {
      const el = els[i];
      if (allowTransitions) {
        el.classList.remove('notransition');
      } else {
        el.classList.add('notransition');
      }
    }
  }
  const toggleGiscusIfUsed = (isAlternate, darkModeDefault) => {
    const baseTheme = document.querySelector('#giscus-base-theme')?.value ?? 'light';
    const alternateTheme = document.querySelector('#giscus-alt-theme')?.value ?? 'dark';
    let newTheme = '';
    if(darkModeDefault) {
      newTheme = isAlternate ? baseTheme : alternateTheme;
    } else {
      newTheme = isAlternate ? alternateTheme : baseTheme;
    }
    const changeGiscusTheme = () => {
      // From: https://github.com/giscus/giscus/issues/336
      const sendMessage = (message) => {
        const iframe = document.querySelector('iframe.giscus-frame');
        if (!iframe) return;
        iframe.contentWindow.postMessage({ giscus: message }, 'https://giscus.app');
      }
      sendMessage({
        setConfig: {
          theme: newTheme
        }
      });
    }
    const isGiscussLoaded = window.document.querySelector('iframe.giscus-frame') !== null;
    if (isGiscussLoaded) {
      changeGiscusTheme();
    }
  }
  const toggleColorMode = (alternate) => {
    // Switch the stylesheets
    const alternateStylesheets = window.document.querySelectorAll('link.quarto-color-scheme.quarto-color-alternate');
    manageTransitions('#quarto-margin-sidebar .nav-link', false);
    if (alternate) {
      enableStylesheet(alternateStylesheets);
      for (const sheetNode of alternateStylesheets) {
        if (sheetNode.id === "quarto-bootstrap") {
          toggleBodyColorMode(sheetNode);
        }
      }
    } else {
      disableStylesheet(alternateStylesheets);
      toggleBodyColorPrimary();
    }
    manageTransitions('#quarto-margin-sidebar .nav-link', true);
    // Switch the toggles
    const toggles = window.document.querySelectorAll('.quarto-color-scheme-toggle');
    for (let i=0; i < toggles.length; i++) {
      const toggle = toggles[i];
      if (toggle) {
        if (alternate) {
          toggle.classList.add("alternate");     
        } else {
          toggle.classList.remove("alternate");
        }
      }
    }
    // Hack to workaround the fact that safari doesn't
    // properly recolor the scrollbar when toggling (#1455)
    if (navigator.userAgent.indexOf('Safari') > 0 && navigator.userAgent.indexOf('Chrome') == -1) {
      manageTransitions("body", false);
      window.scrollTo(0, 1);
      setTimeout(() => {
        window.scrollTo(0, 0);
        manageTransitions("body", true);
      }, 40);  
    }
  }
  const isFileUrl = () => { 
    return window.location.protocol === 'file:';
  }
  const hasAlternateSentinel = () => {  
    let styleSentinel = getColorSchemeSentinel();
    if (styleSentinel !== null) {
      return styleSentinel === "alternate";
    } else {
      return false;
    }
  }
  const setStyleSentinel = (alternate) => {
    const value = alternate ? "alternate" : "default";
    if (!isFileUrl()) {
      window.localStorage.setItem("quarto-color-scheme", value);
    } else {
      localAlternateSentinel = value;
    }
  }
  const getColorSchemeSentinel = () => {
    if (!isFileUrl()) {
      const storageValue = window.localStorage.getItem("quarto-color-scheme");
      return storageValue != null ? storageValue : localAlternateSentinel;
    } else {
      return localAlternateSentinel;
    }
  }
  const darkModeDefault = false;
  let localAlternateSentinel = darkModeDefault ? 'alternate' : 'default';
  // Dark / light mode switch
  window.quartoToggleColorScheme = () => {
    // Read the current dark / light value 
    let toAlternate = !hasAlternateSentinel();
    toggleColorMode(toAlternate);
    setStyleSentinel(toAlternate);
    toggleGiscusIfUsed(toAlternate, darkModeDefault);
  };
  // Ensure there is a toggle, if there isn't float one in the top right
  if (window.document.querySelector('.quarto-color-scheme-toggle') === null) {
    const a = window.document.createElement('a');
    a.classList.add('top-right');
    a.classList.add('quarto-color-scheme-toggle');
    a.href = "";
    a.onclick = function() { try { window.quartoToggleColorScheme(); } catch {} return false; };
    const i = window.document.createElement("i");
    i.classList.add('bi');
    a.appendChild(i);
    window.document.body.appendChild(a);
  }
  // Switch to dark mode if need be
  if (hasAlternateSentinel()) {
    toggleColorMode(true);
  } else {
    toggleColorMode(false);
  }
  const icon = "Óßã";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
<p><span style="color:firebrick">| Produced by</span> <a href="mailto:hello@danielkick.com"><span style="color:firebrick">Daniel Kick, PhD</span></a></p>
</div>   
    <div class="nav-footer-center">
      &nbsp;
    </div>
    <div class="nav-footer-right">
      &nbsp;
    </div>
  </div>
</footer>




</body></html>